<!DOCTYPE html>
<html lang="en" ng-app="plpApplication" xmlns:width="http://www.w3.org/1999/xhtml">
<!--[if lt IE 7]>
<html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>
<html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>
<html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!-->
<html class="no-js"> <!--<![endif]-->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>LINCS Proteomics Landing Page</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="Free HTML5 Template by FREEHTML5.CO"/>
    <meta name="keywords" content="free html5, free template, free bootstrap, html5, css3, mobile first, responsive"/>
    <meta name="author" content="FREEHTML5.CO"/>


        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

    <!--
      //////////////////////////////////////////////////////

      FREE HTML5 TEMPLATE
      DESIGNED & DEVELOPED by FREEHTML5.CO

      Website: 		http://freehtml5.co/
      Email: 			info@freehtml5.co
      Twitter: 		http://twitter.com/fh5co
      Facebook: 		https://www.facebook.com/fh5co

      //////////////////////////////////////////////////////
       -->

    <!-- Facebook and Twitter integration -->
    <meta property="og:title" content=""/>
    <meta property="og:image" content=""/>
    <meta property="og:url" content=""/>
    <meta property="og:site_name" content=""/>
    <meta property="og:description" content=""/>
    <meta name="twitter:title" content=""/>
    <meta name="twitter:image" content=""/>
    <meta name="twitter:url" content=""/>
    <meta name="twitter:card" content=""/>

    <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
    <link rel="shortcut icon"
          href="images/lens4.jpg"/>

    <link href='https://fonts.googleapis.com/css?family=Roboto:400,100,300,600,400italic,700' rel='stylesheet'
          type='text/css'>

    <!-- Animate.css -->
    <link rel="stylesheet" href="css/animate.css">
    <!-- Flexslider -->
    <link rel="stylesheet" href="css/flexslider.css">
    <!-- Icomoon Icon Fonts-->
    <link rel="stylesheet" href="css/icomoon.css">
    <!-- Magnific Popup -->
    <link rel="stylesheet" href="css/magnific-popup.css">
    <!-- Bootstrap  -->
    <link rel="stylesheet" href="css/bootstrap.css">

    <!--
    Default Theme Style
    You can change the style.css (default color purple) to one of these styles
    -->
    <link rel="stylesheet" href="css/style.css">

    <!-- Styleswitcher ( This style is for demo purposes only, you may delete this anytime. ) -->
    <link rel="stylesheet" id="theme-switch" href="css/style.css">
    <!-- End demo purposes only -->


    <style>
        /* For demo purpose only */

        /*
        GREEN
        8dc63f
        RED
        FA5555
        TURQUOISE
        27E1CE
        BLUE
        2772DB
        ORANGE
        FF7844
        YELLOW
        FCDA05
        PINK
        F64662
        PURPLE
        7045FF

        */

        /* For Demo Purposes Only ( You can delete this anytime :-) */
        #colour-variations {
            padding: 10px;
            -webkit-transition: 0.5s;
            -o-transition: 0.5s;
            transition: 0.5s;
            width: 140px;
            position: fixed;
            left: 0;
            top: 100px;
            z-index: 999999;
            background: #fff;
            /*border-radius: 4px;*/
            border-top-right-radius: 4px;
            border-bottom-right-radius: 4px;
            -webkit-box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
            -moz-box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
            -ms-box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
            box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
        }

        #colour-variations.sleep {
            margin-left: -140px;
        }

        #colour-variations h3 {
            text-align: center;;
            font-size: 11px;
            letter-spacing: 2px;
            text-transform: uppercase;
            color: #777;
            margin: 0 0 10px 0;
            padding: 0;;
        }

        #colour-variations ul,
        #colour-variations ul li {
            padding: 0;
            margin: 0;
        }

        #colour-variations ul {
            margin-bottom: 20px;
            float: left;
        }

        #colour-variations li {
            list-style: none;
            display: inline;
        }

        #colour-variations li a {
            width: 20px;
            height: 20px;
            position: relative;
            float: left;
            margin: 5px;
        }

        #colour-variations li a[data-theme="style"] {
            background: #8dc63f;
        }

        #colour-variations li a[data-theme="red"] {
            background: #FA5555;
        }

        #colour-variations li a[data-theme="turquoise"] {
            background: #27E1CE;
        }

        #colour-variations li a[data-theme="blue"] {
            background: #2772DB;
        }

        #colour-variations li a[data-theme="orange"] {
            background: #FF7844;
        }

        #colour-variations li a[data-theme="yellow"] {
            background: #FCDA05;
        }

        #colour-variations li a[data-theme="pink"] {
            background: #F64662;
        }

        #colour-variations li a[data-theme="purple"] {
            background: #7045FF;
        }

        #colour-variations a[data-layout="boxed"],
        #colour-variations a[data-layout="wide"] {
            padding: 2px 0;
            width: 48%;
            border: 1px solid #ededed;
            text-align: center;
            color: #777;
            display: block;
        }

        #colour-variations a[data-layout="boxed"]:hover,
        #colour-variations a[data-layout="wide"]:hover {
            border: 1px solid #777;
        }

        #colour-variations a[data-layout="boxed"] {
            float: left;
        }

        #colour-variations a[data-layout="wide"] {
            float: right;
        }

        .option-toggle {
            position: absolute;
            right: 0;
            top: 0;
            margin-top: 5px;
            margin-right: -30px;
            width: 30px;
            height: 30px;
            background: #8dc63f;
            text-align: center;
            border-top-right-radius: 4px;
            border-bottom-right-radius: 4px;
            color: #fff;
            cursor: pointer;
            -webkit-box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
            -moz-box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
            -ms-box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
            box-shadow: 0 0 9px 0 rgba(0, 0, 0, .1);
        }

        .option-toggle i {
            top: 2px;
            position: relative;
        }

        .option-toggle:hover, .option-toggle:focus, .option-toggle:active {
            color: #fff;
            text-decoration: none;
            outline: none;
        }
    </style>
    <!-- End demo purposes only -->


    <!-- Modernizr JS -->
    <script src="js/modernizr-2.6.2.min.js"></script>
    <!-- FOR IE9 below -->
    <!--[if lt IE 9]>
    <script src="js/respond.min.js"></script>
    <![endif]-->
    <!--This is for the google analytics-->
    <script>
        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
                (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

        ga('create', 'UA-101774139-2', 'auto');
        ga('send', 'pageview');

    </script>

</head>


<!--
    INFO:
    Add 'boxed' class to body element to make the layout as boxed style.
    Example:
    <body class="boxed">
-->
<body>
<div ng-controller="NavigationCtrl as ctrl" ng-init="initNetwork()">

    <!-- Loader -->
    <div class="fh5co-loader"></div>

    <section id="fh5co-page">


        <section id="fh5co-header">
            <div class="container">
                <nav role="navigation">
                    <ul class="pull-left left-menu">
                        <!--	<li><a href="about.html">About</a></li> -->
                        <li ><a href="assay.html">Explore</a></li>

                        <li><a href="download.html" >Download</a></li>
                    </ul>
                    <h1 id="fh5co-logo"><a href="index.html">LINCS PROTEOMICS<span>.</span></a></h1>

                    <ul class="pull-right right-menu">
                        <li><a href="tools.html">Tools</a></li>
                        <li class="active"><a href="about.html">About</a></li>

                    </ul>
                </nav>
            </div>
        </section>
        <!--This is for showing header when scroll down-->
        <div class="topMenu">
            <section id="fh5co-header2">
                <div class="container">
                    <nav role="navigation">
                        <ul class="pull-left left-menu">
                            <!--	<li><a href="about.html">About</a></li> -->
                            <li ><a href="assay.html">Explore</a></li>
                            <li><a href="download.html" >Download</a></li>
                        </ul>
                        <h3 id="fh5co-logo2"><a href="index.html">LINCS PROTEOMICS<span>.</span></a></h3>
                        <ul class="pull-right right-menu">
                            <li><a href="tools.html">Tools</a></li>
                            <li class="active"><a href="about.html">About</a></li>

                        </ul>
                    </nav>
                </div>
            </section>
        </div>


        <!--
                <section id="fh5co-header">
                    <div class="container">
                        <nav role="navigation">
                            <ul class="pull-left left-menu">
                                <li class="active"><a href="about.html">About</a></li>
                                <li><a href="tour.html">Tour</a></li>
                                <li><a href="pricing.html">Pricing</a></li>
                            </ul>
                            <h1 id="fh5co-logo"><a href="index.html">guide<span>.</span></a></h1>
                            <ul class="pull-right right-menu">
                                <li><a href="#">Login</a></li>
                                <li class="fh5co-cta-btn"><a href="#">Sign up</a></li>
                            </ul>
                        </nav>
                    </div>
                </section>-->
        <!-- #fh5co-header -->





        <section id="fh5co-hero" class="no-js-fullheight"
                 style="background-image: url(images/University-of-Cincinnati-903.jpg);" data-next="yes">
            <div class="fh5co-overlay"></div>
            <div class="container">
                <div class="fh5co-intro no-js-fullheight">
                    <div class="fh5co-intro-text">
                        <!--
                            INFO:
                            Change the class to 'fh5co-right-position' or 'fh5co-center-position' to change the layout position
                            Example:
                            <div class="fh5co-right-position">
                        -->
                        <div class="fh5co-center-position">
                            <h2 class="animate-box" style="font-size: 45px">About Us</h2>
                            <h3 class="animate-box">
                                LINCS Proteomics Data Portal enables exploration of LINCS proteomic
                                data and related resources. The portal provides tailored “data-centric”
                                and “assay (or annotation)-centric” views to facilitate navigation, search,
                                and interpretation of LINCS proteomics data.

                            </h3>

                        </div>
                    </div>
                </div>
            </div>
            <div class="fh5co-learn-more animate-box">
                <a href="#" class="scroll-btn">
                    <span class="text">Explore more about us</span>
                    <span class="arrow"><i class="icon-chevron-down"></i></span>
                </a>
            </div>
        </section>
        <!-- END #fh5co-hero -->

        <!--<section id="fh5co-features-3" class="boredr-top" style="padding: 60px 5px 5px 5px">-->
            <!--&lt;!&ndash;<div class="container" >&ndash;&gt;-->
            <!--<div class="row " style="padding: 5px 5px 5px 5px">-->
                <!--<div class="col-md-6 col-md-offset-3 text-center">-->
                    <!--<h2 class="fh5co-lead animate-box">What is LINCS?</h2>-->
                    <!--<p class="fh5co-sub-lead animate-box">The Library of Network-Based Cellular Signatures-->
                        <!--(<a href="http://www.lincsproject.org/" target="_blank">LINCS</a>)-->
                        <!--Program aims to create a network-based understanding of biology by cataloging changes in-->
                        <!--gene-->
                        <!--expression and other cellular processes that occur when cells are exposed to a variety of-->
                        <!--perturbing agents. </p>-->
                <!--</div>-->
            <!--</div>-->

           <!---->
        <!--</section>-->
        <section id="fh5co-about" style="padding: 60px 5px 5px 5px">

            <div class="container">
                <div class="row ">
                    <div class="col-sm-5">
                        <figure class="animate-box">
                            <img src="images/all-to-gether.png" alt="Free HTML5 Bootstrap Template by FREEHTML5.co"
                                 class="img-responsive" width="450px" height="450px">
                        </figure>
                    </div>
                    <div class="col-sm-7">
                        <!--<div class="fh5co-label animate-box">Who We Are:</div>-->
                        <h2 class="fh5co-lead animate-box">LINCS Proteomics Landing Page</h2>
                        <p class="fh5co-sub-lead animate-box">
                            <!--<div class="container" >-->
                            <!--<h2></h2>-->
                            <!--<p >-->

                            LINCS proteomics portal has been developed by the LINCS Data Coordination and Integration Center
                            (DCIC) in collaboration with other LINCS centers, and with the goal of facilitating the
                            navigation through and interaction with the LINCS proteomics data.
                            Your feedback is highly appreciated and please

                            <a
                                    href="mailto:behrouz.shamsaei@uc.edu?subject=Support">contact our team using this link.</a>

                        </p>

                        <!--<hr/>-->
                    </div>
                    <div class="col-sm-12">
                    <h2 class="fh5co-lead animate-box">The LINCS-Proteomics portal aims to:</h2>
                    <p class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear"></i>&nbsp; Facilitate
                        navigation, search, and interpretation of LINCS proteomics data using alternative
                        “data-centric” and “assay-centric” lens or views<br/>

                        <i class="fa fa-gear"></i> &nbsp; Provide a guide
                        to LINCS Data Portal other resources developed by LINCS-DCIC to facilitate finding and
                        downloading proteomic data<br/>

                        <i class="fa fa-gear"></i>&nbsp; Provide intuitive
                        representations of relationship between protein entities that are measured as the snapshot
                        of the biological state provided by each method/assay<br/>

                        <i class="fa fa-gear"></i>&nbsp; Provide links to
                        other resources and tools for proteomics, in particular those developed as part of the LINCS
                        consortium<br/>

                    </div>
                </div>
            </div>
        </section>

        <!--<hr/>-->


        <section id="fh5co-projects2" style="padding: 30px 0px 0px 0px">

            <div class="container">
                <h3 class="fh5co-lead animate-box" style="background-color:#929971">Publications:</h3>
                <div class="fh5co-label animate-box">For complete list of LINCS consortium publications please visit <a
                        href="http://www.lincsproject.org/LINCS/publications" target="_blank">here</a>.
                </div>

                <h2 class="fh5co-lead animate-box" style="background-color:#d1d8ab">Featured LINCS Publications:</h2>

                <div class="fh5co-sub-lead animate-box">
                    <a ng-href="http://www.lincsproject.org/" target="_blank">
                        <img border="0"  src="images/DCIC_BD2K.png" width="7%" height="7%">
                    </a>&nbsp;

                    Keenan AB, et al [...] Pillai A, 2017.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/29199020"
                                                                     target="_blank">The Library of Integrated Network-Based Cellular Signatures NIH program: System-level cataloging of human cells response to perturbations.</a> Cell Systems. pii: S2405-4712(17):30490-30498.
                    <br/>
                    <hr/>
                </div>



                <div class="fh5co-sub-lead animate-box">
                    <a ng-href="http://lincsportal.ccs.miami.edu/dcic-portal/" target="_blank">
                        <img border="0"  src="images/ldp_logo.png" width="5%" height="5%">
                    </a>&nbsp;


                    Koleti A, Terryn R, Stathias V, Chung C, Cooper DJ, Turner JP, Vidovic D, Forlin M, Kelley TT, D'Urso A, Allen BK, Torre D, Jagodnik KM, Wang L, Jenkins SL, Mader C, Niu W, Fazel M, Mahi M, Pilarczyk M, Clark N, Shamsaei B, Meller J, Vasiliauskas J, et al. 2017. <a href="https://academic.oup.com/nar/article/46/D1/D558/4621325"
                                                                     target="_blank">Data Portal for the Library of Integrated Network-based Cellular Signatures (LINCS) program: integrated access to diverse large-scale cellular perturbation response data.</a> Nucleic Acids Research. gkx1063.<br/>
                    <hr/>
                </div>



                <h2 class="fh5co-lead animate-box" style="background-color:#d1d8ab">Featured LINCS Proteomics Publications:</h2>

                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Koplev S, Lin K, Dohlman AB, Ma'ayan A. 2018. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29293502"
                                                                     target="_blank">Integration of pan-cancer
                    transcriptomics with RPPA proteomics
                    reveals mechanisms of epithelial-mesenchymal transition.</a> PLoS Computational Biology.
                    14(1):e1005911.<br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Erickson BK, Rose CM, Braun CR, Erickson AR, Knott J, McAlister GC, Wühr M, Paulo JA, Everley RA,
                    Gygi
                    SP. 2017. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28065596" target="_blank">A strategy to
                    combine
                    sample multiplexing with targeted proteomics assays for high-throughput protein signature
                    characterization.</a> Molecular Cell. 65(2):361-370.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Litichevskiy L, Peckner R, Abelin JG, Asiedu JK, Creech AL, Davis JF, Davison D, Dunning CM,
                    Egertson
                    JD, Egri S, Gould J, et al [...] Jaffe J. 2017. <a
                        href="https://www.biorxiv.org/content/early/2017/09/07/185918" target="_blank">A library of
                    phosphoproteomic and chromatin signatures for characterizing cellular responses to drug
                    perturbations.</a> bioRxiv.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Everley RA, Huttlin EL, Erickson AR, Beausoleil SA, Gygi SP. 2016. <a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27978624" target="_blank">Neutral loss is a very
                    common
                    occurrence in phosphotyrosine-containing peptides labeled with isobaric tags.</a> Journal of
                    Proteome
                    Research. 16(2):1069-1076.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Parker SJ, Venkatraman V, Van Eyk JE. 2016. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27432805"
                                                                   target="_blank">Effect of peptide assay library size
                    and
                    composition in targeted data-independent acquisition-MS analyses.</a> Proteomics. 16(15-16):2221-37.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Shi T, Niepel M, McDermott JE, Gao Y, Nicora CD, Chrisler WB, Markillie LM, Petyuk VA, Smith RD,
                    Rodland
                    KD, Sorger PK, Qian WJ, Wiley HS. 2016. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27405981"
                                                               target="_blank">Conservation of protein abundance
                    patterns
                    reveals the regulatory architecture of the EGFR-MAPK pathway.</a> Science Signaling. 9(436):rs6.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG, Ma'ayan A. 2016. <a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27374120" target="_blank">The Harmonizome: a
                    collection of
                    processed datasets gathered to serve and mine knowledge about genes and proteins.</a> Database
                    (Oxford).
                    baw100.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Zhang YJ, Gendron TF, Grima JC, Sasaguri H, Jansen-West K, Xu YF, Katzman RB, Gass J, Murray ME,
                    Shinohara M, Lin WL, Garrett A, Stankowski JN, Daughrity L, Tong J, Perkerson EA, Yue M, Chew J,
                    Castanedes-Casey M, Kurti A, Wang ZS, Liesinger AM, Baker JD, Jiang J, Lagier-Tourenne C, Edbauer D,
                    Cleveland DW, Rademakers R, Boylan KB, Bu G, Link CD, Dickey CA, Rothstein JD, Dickson DW, Fryer JD,
                    Petrucelli L. 2016.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26998601" target="_blank"> C9ORF72
                    poly(GA) aggregates sequester and impair HR23 and nucleocytoplasmic transport proteins.</a> Nature
                    Neuroscience. 19(5):668-77.

                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;
                    Abelin JG, Patel J, Lu X, Feeney CM, Fagbami L, Creech AL, Hu R, Lam D, Davison D, Pino L, Qiao JW,
                    Kuhn
                    E, Officer A, Li J, Abbatiello S, Subramanian A, Sidman R, Snyder E, Carr SA, Jaffe JD. 2016.<a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26912667" target="_blank"> Reduced-representation
                    phosphosignatures measured by quantitative targeted MS capture cellular states and enable
                    large-scale
                    comparison of drug-induced phenotypes.</a> Molecular and Cellular Proteomics. 15(5):1622-41.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Holewinski RJ, Parker SJ, Matlock AD, Venkatraman V, Van Eyk JE. 2016.<a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26867750" target="_blank"> Methods for SWATH: Data
                    independent acquisition on TripleTOF mass spectrometers.</a> Methods in Molecular Biology.
                    1410:265-79.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Fuller HR, Mandefro B, Shirran SL, Gross AR, Kaus AS, Botting CH, Morris GE, Sareen D. 2016.<a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/26793058" target="_blank"> Spinal muscular atrophy
                    patient
                    iPSC-derived motor neurons have reduced expression of proteins important in neuronal
                    development.</a>
                    Frontiers in Cellular Neuroscience. 9:506.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;
                    van Hasselt JG, Hansen J, Xiong Y, Shim J, Pickard A, Jayaraman G, Hu B, Mahajan M, Gallo J, Sobie
                    EA,
                    Birtwistle MR, Azeloglu EU, Iyengar R. 2016.<a
                        href="https://www.biorxiv.org/content/early/2016/09/19/075754" target="_blank">
                    Clinically-weighted
                    transcriptomic signatures for protein kinase inhibitor associated cardiotoxicity.</a> bioRxiv.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Wang Z, Clark NC, Ma'ayan A. 2015.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26048415"
                                                         target="_blank"> Dynamics of the discovery process of
                    protein-protein interactions from low content studies.</a> BMC Systems Biology. 9(1):26.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;
                    Hornbeck PV, Zhang B, Murray B, Kornhauser JM, Latham V, Skrzypek E. 2015. <a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/25514926" target="_blank">PhosphoSitePlus, 2014:
                    mutations, PTMs and recalibrations.</a> Nucleic Acids Research. 43(Database issue):D512-20.
                    <br/>
                    <hr/>
                </div>

                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Sharma V, Eckels J, Taylor GK, Shulman NJ, Stergachis AB, Joyner SA, Yan P, Whiteaker JR, Halusa GN,
                    Schilling B, Gibson BW, Colangelo CM, Paulovich AG, Carr SA, Jaffe JD, MacCoss MJ, MacLean B.
                    2014.<a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/25102069" target="_blank"> Panorama: a targeted
                    proteomics
                    knowledge base.</a> Journal of Proteome Research. 13(9):4205-10.
                    Everley RA, Huttlin EL, Erickson AR, Beausoleil SA, Gygi SP. 2016. <a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/27978624" target="_blank">Neutral loss is a very
                    common
                    occurrence in phosphotyrosine-containing peptides labeled with isobaric tags.</a> Journal of
                    Proteome
                    Research. 16(2):1069-1076.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    McAllister FE, Niepel M, Haas W, Huttlin E, Sorger PK, Gygi SP. 2013. <a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/23607489" target="_blank">Mass spectrometry based
                    method
                    to increase throughput for kinome analyses using ATP probes.</a> Analytical Chemistry.
                    85(9):4666-74.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Luckert K, Gujral TS, Chan M, Sevecka M, Joos TO, Sorger PK, Macbeath G, Pötz O. 2012.<a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/22234610" target="_blank"> A dual array-based approach
                    to
                    assess the abundance and posttranslational modification state of signaling proteins.</a> Science
                    Signaling. 5(206):1.
                    <br/>
                    <hr/>
                </div>


                <div class="fh5co-sub-lead animate-box">
                    <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                    Kirouac DC, Saez-Rodriguez J, Swantek J, Burke JM, Lauffenburger DA, Sorger PK. 2012.<a
                        href="https://www.ncbi.nlm.nih.gov/pubmed/22548703" target="_blank"> Creating and analyzing
                    pathway
                    and protein interaction compendia for modelling signal transduction networks.</a> BMC Systems
                    Biology.
                    6:29.
                    <br/>
                    <hr/>
                </div>


            </div>

        </section>

        <!--<div class="container">-->


                <!--<div class="row">-->
                <!--<div class="col-sm-3">-->
                <!--<figure class="animate-box">-->
                <!--<img src="images/pinet-logo.png" alt="Free HTML5 Bootstrap Template by FREEHTML5.co"-->
                <!--class="img-responsive" width="300" height="300">-->
                <!--</figure>-->
                <!--</div>-->
                <!--<div class="col-sm-9">-->
                <!--<div class="fh5co-label animate-box">More about piNET</div>-->
                <!--<h2 class="fh5co-lead animate-box">piNET</h2>-->
                <!--<p class="fh5co-sub-lead animate-box"><a href="http://eh3.uc.edu/pinet/" target="_blank">-->
                <!--<span style="color:blue">&#10004;</span> piNet</a> can be used to annotate, map and-->
                <!--analyze-->
                <!--a set of peptide moieties-->
                <!--(including post-translationally modified peptides) identified by either-->
                <!--targeted (SRM type) or SWATH (discovery) proteomic assays, or another-->
                <!--related mass spectrometry based protein moieties identification and-->
                <!--quantification technique. Starting from a set of peptides, and optional-->
                <!--modifications within those peptides, piNET provides mapping into-->
                <!--canonical UniProt protein entries, standardized annotation of-->
                <!--modifications using PsiMOD ontology, and mapping of the sites of PTMs in-->
                <!--the full UniProt entries. piNET further generates Protein Line Notation-->
                <!--tokens to represent each peptide moiety with modifications for automated-->
                <!--annotation exchange systems. The set of proteins and PTM sites can be-->
                <!--further projected onto biological pathways using enrichment analysis and-->
                <!--a comprehensive library of biological gene/protein sets through Enrichr.-->
                <!--For phosphorylation, PhosphoSitePlus known kinase - target peptides (and-->
                <!--extrapolations form known examples to sufficiently similar putative-->
                <!--target peptides) are used to provide a site specific network and-->
                <!--pathways views. Work is in progress to extend the capabilities to-->
                <!--include other PTMs in network projections, and to enable annotation of-->
                <!--mouse and rat proteomics data.</p>-->
                <!--</div>-->
                <!--</div>-->


                <!--<div class="row">-->
                <!--<div class="col-sm-3">-->
                <!--<figure class="animate-box">-->
                <!--<img src="images/pilincs.jpg" alt="Free HTML5 Bootstrap Template by FREEHTML5.co"-->
                <!--class="img-responsive" width="350" height="350">-->
                <!--</figure>-->
                <!--</div>-->
                <!--<div class="col-sm-9">-->
                <!--<div class="fh5co-label animate-box">More about piLINCS</div>-->
                <!--<h2 class="fh5co-lead animate-box">piLINCS</h2>-->
                <!--<p class="fh5co-sub-lead animate-box"><a href="http://eh3.uc.edu/pilincs/#/"-->
                <!--target="_blank">-->
                <!--<span style="color:blue">&#10004;</span> piLINCS</a> has been developed as part of the-->
                <!--BD2K-LINCS Data Coordination and Integration Center-->
                <!--to provide interactive and API access to proteomics data generated by the Library of-->
                <!--Integrated-->
                <!--Network-based Signatures (LINCS) program. At present, piLINCS enables access to the-->
                <!--reduced-->
                <!--representation phosphoproteomic ("P100") and global chromatin ("GCP") profiles generated-->
                <!--by-->
                <!--using-->
                <!--mass spectrometry techniques to characterize proteome level molecular signatures of-->
                <!--(responses to)-->
                <!--small molecule and genetic perturbations in a number of different cell lines-->
                <!--(for details see LINCS/Broad). These different profiles (obtained by merging QC+NORM-->
                <!--level-->
                <!--processed GCT files from Panorama repository) can be browsed interactively, filtered-->
                <!--based-->
                <!--on user-->
                <!--defined selection of cell - perturbation - dose tuples, merged across different profiles-->
                <!--(for the-->
                <!--same tuples), analyzed in iLINCS (more tools to come), and exported in several different-->
                <!--formats.</p>-->
                <!--</div>-->
                <!--</div>-->


                <!--</div>-->




        <div id="fh5co-testimonials2" class="boredr-top">
            <div class="container">


                <!--<h2 class="fh5co-lead animate-box text-center">LINCS Proteomics Centers</h2>-->

                <div class="row ">
                    <div class="col-md-6 col-md-offset-3 text-center">

                        <h2 class="fh5co-lead animate-box" style="font-size: 30px">LINCS Proteomics Data
                            Centers </h2>
                        <div class="fh5co-label animate-box">This list includes data generating and data
                            processing centers
                        </div>
                    </div>
                </div>
                <div class="row">
                    <div class="col-md-8 col-md-offset-2 text-center animate-box">
                        <div class="flexslider">
                            <ul class="slides">
                                <li>
                                    <blockquote>
                                        <h3>LINCS PCCSE </h3>
                                        <p>The LINCS Proteomic Characterization Center for Signaling and
                                            Epigenetics at
                                            the
                                            Broad Institute/MIT/University of Washington utilizes the P100 and
                                            GCP
                                            assays to
                                            profile the response of human cell lines to perturbations.</p>
                                        <p><cite>&nbsp;</cite></p>
                                    </blockquote>
                                </li>
                                <li>
                                    <blockquote>
                                        <h3>LINCS DToxS </h3>
                                        <p>The Drug Toxicity Signature Generation Center is a Systems
                                            Pharmacology
                                            research
                                            center at the Icahn School of Medicine. The proteomics experiments
                                            are
                                            conducted at
                                            the Center for Advanced Proteomics, Rutgers-New Jersey Medical
                                            School.</p>
                                        <p><cite>&nbsp;</cite></p>
                                    </blockquote>
                                </li>
                                <li>
                                    <blockquote>
                                        <h3>HMS LINCS </h3>
                                        <p>The HMS LINCS Center collects multiplex perturbagen-response data
                                            across a
                                            number of complementary
                                            experimental platforms including live- and fixed-cell imaging,
                                            sandwich
                                            immunoassays, protein mass
                                            spectrometry, and mRNA profiling as a means to measure the levels,
                                            localization, and states of modification.</p>
                                        <p><cite>&nbsp;</cite></p>
                                    </blockquote>
                                </li>
                                <li>
                                    <blockquote>
                                        <h3>NeuroLINCS </h3>
                                        <p>The NeuroLINCS Center studies ALS and SMA by characterizing the
                                            molecular
                                            networks within
                                            patient-derived iPSCs, and their differentiated progeny, across many
                                            regulatory layers.</p>
                                        <p><cite>&nbsp;</cite></p>
                                    </blockquote>
                                </li>
                                <li>
                                    <blockquote>
                                        <h3>BD2K-LINCS DCIC </h3>
                                        <p>The BD2K-LINCS Data Coordination and Integration Centers at Icahn
                                            School of
                                            Medicine, university of Miami
                                            and university of Cincinnati are responsible for harmonizing
                                            LINCS data with other available resources.</p>
                                        <p><cite>&nbsp;</cite></p>
                                    </blockquote>
                                </li>
                            </ul>
                        </div>
                        <div class="flexslider-controls" style="padding-top: 10px">
                            <ol class="flex-control-nav">
                                <li class="animate-box"><img src="images/labkey-logo2.png"
                                                             alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                </li>
                                <li class="animate-box"><img src="images/dtoxs2.png"
                                                             alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                </li>
                                <li class="animate-box"><img src="images/hms-lincs2.png"
                                                             alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                </li>
                                <li class="animate-box"><img src="images/neurolincs2.jpeg"
                                                             alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                </li>
                                <li class="animate-box"><img src="images/dcic3.png"
                                                             alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                </li>
                            </ol>
                        </div>
                    </div>
                </div>

                <!--<div id="fh5co-pricing">-->
                    <div class="container" style="padding-top: 30px">

                        <iframe src="https://www.google.com/maps/d/embed?mid=1MhDhNDsvCJvAmSNypmyCaj-g2EQ&hl=en"
                                width="100%"
                                height="700"></iframe>
                        <!--</p>-->
                    </div>
                    <!--<h3 class="animate-box"></h3>-->
                    <!--<hr/>-->


                <!--</div>-->
            </div>
        </div>
        <!-- END #fh5co-subscribe -->


        <footer id="fh5co-footer">
            <div class="container">



                    <div class="col-md-5 col-sm-6 col-xs-10 animate-box">
                        <div class="fh5co-footer-widget">
                            <!--<h3>Company</h3>-->
                            <ul class="fh5co-links">
                                <li class="animate-box"><img src="images/footnote.jpg"
                                                             alt="Free HTML5 Bootstrap Template by FREEHTML5.co"></li>
                            </ul>
                        </div>
                    </div>




            </div>


            <div class="fh5co-copyright animate-box">
                <div class="container">
                    <div class="row">
                        <div class="col-md-12">

                            <p class="fh5co-left">

                                <small>&copy; 2017 <a href="http://www.lincsproject.org/">LINCS Program. All rights
                                    reserved.</a></small>
                            </p>
                            <p class="fh5co-right">
                                <small class="fh5co-right">Developed within NIH funded <a
                                        href="http://www.lincsproject.org/" target="_blank">LINCS</a> project.
                                </small>
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </footer>
        <!-- END #fh5co-footer -->
    </section>
</div>
<!-- END #fh5co-page -->

<!-- For demo purposes Only ( You may delete this anytime :-)
<div id="colour-variations">
    <a class="option-toggle"><i class="icon-gear"></i></a>
    <h3>Preset Colors</h3>
    <ul>
        <li><a href="javascript: void(0);" data-theme="style"></a></li>
        <li><a href="javascript: void(0);" data-theme="red"></a></li>
        <li><a href="javascript: void(0);" data-theme="turquoise"></a></li>
        <li><a href="javascript: void(0);" data-theme="blue"></a></li>
        <li><a href="javascript: void(0);" data-theme="orange"></a></li>
        <li><a href="javascript: void(0);" data-theme="yellow"></a></li>
        <li><a href="javascript: void(0);" data-theme="pink"></a></li>
        <li><a href="javascript: void(0);" data-theme="purple"></a></li>
    </ul>
    <a href="#" data-layout="boxed">Boxed</a>
    <a href="#" data-layout="wide">Wide</a>
</div> -->
<!-- End demo purposes only -->


<!-- jQuery -->
<script src="js/jquery.min.js"></script>
<!-- jQuery Easing -->
<script src="js/jquery.easing.1.3.js"></script>
<!-- Bootstrap -->
<script src="js/bootstrap.min.js"></script>
<!-- Waypoints -->
<script src="js/jquery.waypoints.min.js"></script>
<!-- Flexslider -->
<script src="js/jquery.flexslider-min.js"></script>
<!-- Magnific Popup -->
<script src="js/jquery.magnific-popup.min.js"></script>
<script src="js/magnific-popup-options.js"></script>

<!-- For demo purposes only styleswitcher ( You may delete this anytime ) -->
<script src="js/jquery.style.switcher.js"></script>
<script>
    $(function () {
        $('#colour-variations ul').styleSwitcher({
            defaultThemeId: 'theme-switch',
            hasPreview: false,
            cookie: {
                expires: 30,
                isManagingLoad: true
            }
        });
        $('.option-toggle').click(function () {
            $('#colour-variations').toggleClass('sleep');
        });
    });
</script>
<!-- End demo purposes only -->

<!-- Main JS (Do not remove) -->
<script src="js/main.js"></script>

<!--
INFO:
jQuery Cookie for Demo Purposes Only.
The code below is to toggle the layout to boxed or wide
-->
<script src="js/jquery.cookie.js"></script>
<script>
    $(function () {

        if ($.cookie('layoutCookie') != '') {
            $('body').addClass($.cookie('layoutCookie'));
        }

        $('a[data-layout="boxed"]').click(function (event) {
            event.preventDefault();
            $.cookie('layoutCookie', 'boxed', {expires: 7, path: '/'});
            $('body').addClass($.cookie('layoutCookie')); // the value is boxed.
        });

        $('a[data-layout="wide"]').click(function (event) {
            event.preventDefault();
            $('body').removeClass($.cookie('layoutCookie')); // the value is boxed.
            $.removeCookie('layoutCookie', {path: '/'}); // remove the value of our cookie 'layoutCookie'
        });
    });
</script>

<script>
    $(document).ready(function () {
        $('[data-toggle="tooltip"]').tooltip();
    });
</script>

<!-- AngularJS -->
<script src="bower_components/angular/angular.min.js"></script>
<!--<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.2.0rc1/angular-route.min.js"></script>-->
<script src="bower_components/angular-route/angular-route.min.js"></script>
<!-- Bootstrap JS -->
<script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>

<!-- Animation/Autocomplete -->
<!--<script src="https://cdn.jsdelivr.net/angularjs/1.5.5/angular-animate.min.js" data-semver="1.5.5"></script>-->
<!--<script src="https://cdn.jsdelivr.net/angularjs/1.5.5/angular-sanitize.min.js" data-semver="1.5.5"></script>-->
<!---->


<script src="bower_components/https/angular-strap.js" data-semver="v2.3.8"></script>
<script src="bower_components/https/angular-strap.tpl.js" data-semver="v2.3.8"></script>
<script src="bower_components/https/angular-strap.docs.tpl.js" data-semver="v2.3.8"></script>

<!--<script src="https://mgcrea.github.io/angular-strap/dist/angular-strap.js" data-semver="v2.3.8"></script>-->
<!--<script src="https://mgcrea.github.io/angular-strap/dist/angular-strap.tpl.js" data-semver="v2.3.8"></script>-->
<!--<script src="https://mgcrea.github.io/angular-strap/docs/angular-strap.docs.tpl.js" data-semver="v2.3.8"></script>-->

<!--<script src="https://cdnjs.cloudflare.com/ajax/libs/angular-strap/v2.3.8/angular-strap.min.js"></script>-->
<!--<script src="https://cdnjs.cloudflare.com/ajax/libs/angular-strap/v2.3.8/angular-strap.tpl.min.js"></script>-->

<!-- dirPagination JS -->
<script src="bower_components/bootstrap/dist/js/dirPagination.js"></script>


<!--<script src="https://cdnjs.cloudflare.com/ajax/libs/angular-filter/0.5.8/angular-filter.min.js"></script>-->
<script src="bower_components/https/angular-filter.min.js"></script>

<!--d3js-->
<!--<script src="https://d3js.org/d3-selection.v1.min.js"></script>-->
<script src="http://d3js.org/d3.v3.js"></script>
<script src="bower_components/d3/d3.legend.js"></script>

<!-- PLP js -->

<!-- Ng-tags-input-->
<script src="bower_components/ng-tags-input/ng-tags-input.min.js"></script>

<!-- Main JS (Do not remove) -->
<script src="js/app.js"></script>
<script src="js/controllers.js"></script>
<!--<script src="js/main.js"></script>-->

</div>
</body>
</html>

