<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>
<div class="container ">

    <div class="tab-pane fade in active" id="P100">
        <!--<h3>HOME</h3>-->
        <p class="fh5co-sub-lead ">   <a href="" ng-click="scrollTo('P100Info')">P100</a>   has been designed to measure the phosphorylation state of
            cellular
            systems, using a set
            of about hundred pospho-peptide probes whose abundance levels
            are
            measured
            using mass spectrometry
            techniques. P100 can be classified as a 'targeted proteomics'
            assay that
            provides a snapshot of cellular
            signaling cascades that involve phosphorylation of the P100
            target
            sites/peptides.
            <a href="https://panoramaweb.org/labkey/project/LINCS/begin.view?"
               target="_blank">More
                ...</a>
        </p>
    </div>
    <div id="GCP" class="tab-pane fade">
        <!--<h3>Menu 1</h3>-->
        <p class="fh5co-sub-lead ">
            GCP has been designed to measure the histone
            modification state, and thus indirectly the chromatin state of
            cellular
            systems, using a set of about sixty target modified (acetylated,
            methylated
            etc.)
            histone peptides whose abundance levels are measured
            using mass spectrometry.
            <a href="https://panoramaweb.org/labkey/project/LINCS/begin.view?"
               target="_blank">More
                ...</a>
        </p>
    </div>
    <div id="DToxS" class="tab-pane fade">
        <!--<h3>Menu 2</h3>-->
        <p class="fh5co-sub-lead ">
            Mass spectrometry is a sensitive technique that
            measures the mass-to-charge ratio of ions to
            identify and quantify molecules in simple and
            complex mixtures. Samples are loaded into the mass spectrometer
            in
            liquid,
            gas or
            dried form
            and then vaporized and ionized by the ion source. The charge
            that these
            molecules
            receive allows
            the mass spectrometer to accelerate the ions throughout the
            remainder of
            the
            system.
            The ions
            encounter electric and/or magnetic fields from mass analyzers,
            which
            deflect
            the
            paths of individual
            ions based on their mass and charge (m/z). Ions that have
            successfully
            been
            deflected by the mass
            analyzers then hit the ion detector that registers the number of
            ions at
            each m/z
            value.
        </p>
    </div>
    <div id="SWATH" class="tab-pane fade">
        <!--<h3>Menu 3</h3>-->
        <p class="fh5co-sub-lead ">
            SWATH (Sequential Window Acquisition of all THeoretical
            fragment ion spectra) is a new strategy for high
            throughput, label-free protein quantification. It generates
            global,
            quantitative
            protein maps using
            data-independent acquisition of collision-induced dissociation
            (CID)
            spectra
            of all
            precursor ions.
            As a data-independent acquisition (also referred to as MS/MSall)
            SWATH-MS
            has a
            greater coverage of
            peptide identification compared to classical discovery
            approaches.
            Using known fingerprints of target peptides comprising precursor
            mass,
            chromatographic retention
            time and MRM transitions SWATH protein maps can be interrogated
            for
            targeted
            quantification of proteins
            of interest based on high resolution MRM-like signatures. SWATH
            acquires
            all
            MRM
            transitions of all
            precursors and thus does not require tedious assay development
            and
            allows
            for a more
            dynamic data
            interpretation compared to classical MRM experiments.
        </p>
    </div>
    <div id="RPPA" class="tab-pane fade">
        <!--<h3>Menu 3</h3>-->

        <p class="fh5co-sub-lead ">
            Reverse phase protein array (RPPA) is a high-throughput
            technology that performs protein assays on
            thousands of samples simultaneously, including tissue and cell
            lysates,
            serum,
            plasma or other
            body fluids that measure the levels of protein expression, as
            well as
            protein
            modifications such
            as phosphorylation. Protein lysates are arrayed as microspots on
            nitrocellulized
            coated glass slides
            and probed with highly specific antibodies that have been
            validated for
            RPPA. When
            specific antibodies
            are used, RPPA can generate 1,000 times more data points using
            10,000
            times
            less
            sample volume than
            an ordinary western blot, enabling researchers to monitor
            quantitative
            proteomic
            responses for various
            time-scale and input-dose gradients simultaneously.
        </p>
    </div>












    <div style="padding-top: 15px;">
        <h2 class="fh5co-lead " style="background-color:#d1d8ab">Glossary of related definitions:</h2>



        <a name="2-D-PAGE"/>
        <div style="background-color:#eef2d7">

            <h3 class="onerow" style="padding-bottom: 0px">2-D PAGE (Two-dimensional polyacrylamide gel electrophoresis)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>

        </div>

        <p class="fh5co-sub-lead ">
            2D-PAGE is a separation method for gel-based proteomics (also known as classical proteomic approach). This method separates complex mixture of proteins based on molecular mass and charge, and can resolve hundreds to thousands intact proteins in a single gel. After 2D-PAGE, protein identification is usually performed by in-gel tryptic digestion and mass spectrometry.
        </p>
        <h2 style="text-decoration: underline;">Technique:</h2>
        <p class="fh5co-sub-lead ">
            2-D PAGE has two steps of protein separation. First, proteins in a complex sample are separated by their molecular charges (or isoelectric points; pI) in an IPG-strip (a precast gel strip with internal pI gradient) using isoelectric focusing. Second, after isoelectric focusing, the separated proteins are resolved in SDS-PAGE by their molecular weights. This two-step separation results in a 2-D map of numerous protein spots. Ideally, one protein spot contains one proteoform. For protein identification, protein spots of interest are picked up and subjected to in-gel tryptic digestion. The digested peptides can be analyzed by mass spectrometry to get their peptide sequences.
        </p>
        <!--<img src="paris.jpg" alt="Paris" class="center">-->
        <img  src="/lincsproteomics/images/1.png"
              class="centered">
        <h2 style="text-decoration: underline;">Application:</h2>
        <p class="fh5co-sub-lead ">
            2D-PAGE is considered a great tool for detecting proteins with potentially changed post-translational modifications, i.e., phosphorylation and glycosylation. Phosphorylation adds negative charges to a protein, causing a shift of phosphorylated protein to more acidic pI. Glycosylation adds glycan branches to a protein, resulting in a shift of molecular size.
        </p>
        <h2 style="text-decoration: underline;">Limitations: </h2>
        <p class="fh5co-sub-lead ">
            Although gel-based proteomics has very simple analytical workflow, as a front-end separation strategy, 2-D PAGE is a laborious and time-consuming procedure which are disadvantages for large-scale proteomic projects. One needs a period of training to master this procedure and to achieve an acceptable level of result reproducibility. In addition, depth of proteome coverage from a single analysis is much lower than mass spectrometric-based proteomics. All these limitations lead to decreased popularity of 2-D PAGE and gel-based proteomics in the field.
        </p>
        <p style="font-weight: 500">
            Ref: May C, et al., Methods Mol Biol. 2012;893:37-46. PMID: 22665292
        </p>



        <a name="AspN"/>
        <div style="background-color:#eef2d7">

            <h3 class="onerow" style="padding-bottom: 0px">AspN</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>

        </div>

        <p class="fh5co-sub-lead ">
            A metalloprotease enzyme that cleaves amino side of aspartic acid residues.
        </p>



        <a name="b-and-y-ion-series"/>
        <div style="background-color:#eef2d7">

            <h3 class="onerow" style="padding-bottom: 0px">b- and y-ion series</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>

        </div>

        <p class="fh5co-sub-lead ">
            Refers to fragment ions derived from peptide breakage at the peptide bonds during mass spectrometric analysis. The b- and y-ions are indicated if the charge retained on the N-terminus and C-terminus fragments, respectively.
        </p>
        <img  src="/lincsproteomics/images/2.png" class="centered">
        <p class="fh5co-sub-lead ">
            The mass differences obtained from the b- and y-ion series of a peptide can be used to interpret its amino acid sequence.
        </p>
        <p style="font-weight: 500">
            Ref: Marcotte EM. Nat Biotechnol. 2007 Jul;25(7):755-7. PMID: 17621303
        </p>


        <a name="CAD"/>
        <div style="background-color:#eef2d7">

            <h3 class="onerow" style="padding-bottom: 0px">CAD</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>

        </div>

        <p class="fh5co-sub-lead ">
            Refers to collisional activation dissociation, a synonym of CID.
        </p>



        <a name="Bottom-up-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Bottom-up proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead ">
            Refers to the characterization of proteins by analysis of proteolytic peptides. Of several proteases, trypsin has become a gold standard for protein digestion in bottom-up analysis. When bottom-up proteomics is performed on a mixture of proteins in complex samples for discovery purposes, it is called shotgun proteomics. See also “Top-down proteomics” for comparison.
        </p>
        <p style="font-weight: 500">
            Ref: Zhang Y, et al. Chem Rev. 2013 Apr 10;113(4):2343-94. PMID: 23438204
        </p>



        <a name="Carbamidomethylation-(CAM)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Carbamidomethylation (CAM)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead ">
            CAM is a chemical modification introduced to cysteine residues by alkylation with iodoacetamide during proteomic sample preparation.
        </p>



        <a name="Chemical-proteomics-(also-known-as-chemoproteomics)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Chemical proteomics (also known as chemoproteomics)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead ">
            Refers to a research area at the interface between chemistry, biochemistry and cell biology that focused on the mechanisms of actions of small molecular compounds i.e., the mapping of their interacted proteins and their effects on protein expression and post-translational modification in target cells or tissue of interest. There are two main approaches; first, global chemoproteomic profiling to evaluate the cellular response to small molecule treatment in a global view; and second, targeted chemoproteomic approach which adopted affinity-based techniques to determine protein binding profiles of small molecular drug
        </p>
        <p style="font-weight: 500">
            Ref: Bantscheff M. Methods Mol Biol. 2012;803:3-13. PMID: 22065214
        </p>



        <a name="Clinical-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Clinical proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the application of proteomic analysis with the aim of solving a specific clinical problem within the context of a clinical study. The value of clinical proteomics lies in its application with the ultimate goal to improve patient care, for example, detecting biomarkers for early diagnosis, classification for patient-tailored therapy, evaluation of therapeutic response, etc. Accordingly, clinical proteomics should follow the principles and rules of clinical trials/studies, when proteomics should be used to obtain biomarker outcomes. Recommended steps for clinical proteomics are shown as follows:
        </p>



        <a name="Recommended-steps-for-clinical-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Clinical proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>
        <h3>Recommended steps for clinical proteomics</h3>
        <p class="fh5co-sub-lead">
            1) Define a clear clinical question and how the outcome of the study would improve the diagnosis and/or treatment of the disease.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            2) Define the patient and control populations, clinical data to be collected, as well as protocols for sampling and sample preparation.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            3) Define the type of samples needed for the discovery and validation phases.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            4) Define and validate the analytical platforms for discovery (those for validation may well differ).<br/>
            <!--</p >-->
            <!--<p class="fh5co-sub-lead">-->
            5) Obtain IRB approval and written informed consent from the participant.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            6) Perform a pilot study on a validated discovery platform.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            7) Statistically evaluate data from the pilot study to calculate the number of cases and controls for the training set.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            8) Perform study of the training set on the validated platform based on the calculated number of cases and controls.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            9) Evaluate findings from the training set on blinded samples.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            10) Deposit datasets in a public database.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            11) Using these results, transfer the assay to the application platform and test using a training (if applicable) and subsequently a blinded set.<br/>
            <!--</p>-->
            <!--<p class="fh5co-sub-lead">-->
            12) Apply towards clinical use to show whether the findings improve the current clinical situation.<br/>
        </p>
        <p style="font-weight: 500">
            Ref: Mischak H, et al. Proteomics Clin. Appl. 2007, 1, 148–156. PMID: 21136664.<br/>
        </p>



        <a name="CE"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">CE</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to capillary electrophoresis, an alternative to chromatography for peptide and intact protein separation prior to mass spectrometric analysis.
        </p>



        <a name="Collision-induced-dissociation-(CID)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Collision induced dissociation (CID)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Dissociation of precursor ions into fragment ions by collision with gas molecules within the collision cell of mass spectrometer.
        </p>
        <img  src="/lincsproteomics/images/3.png"
              class="centered">
        <!--<img border="0" src="/lincsproteomics/images/3.png" width="4%" height="4%"-->
        <!--style="align-content: center; ">-->



        <a name="Cycle-time"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Cycle time</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the sum of the dwell times of all analyte ions in a given set.
        </p>


        <a name="Data-dependent-acquisition-(DDA)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Data-dependent acquisition (DDA)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a process driven by an automated data acquisition that trigger MS/MS scan depending on the signal intensity (or abundance) of the precursor (or MS1) ions. A subset of the most abundant precursor ions reaching the mass spectrometers detector during MS1 scan are individually isolated and fragmented in sequential MS/MS scan and can be analyzed with a database search algorithm. Instrument setting up and data analysis of DDA is simple, while this method typically yields 1000s of protein identifications. These advantages make DDA the preferred LC-MS/MS method among the wider scientific community. However, DDA method analyzes only a limited number of all precursor ions (identifying the most abundant peptides but missing the rest) and therefore has poor reproducibility of the detection and measurement of low-abundance proteins in complex samples. Run-to-run peptide identification overlap for a given sample is approximately 60%. DDA using a label-free approach has low precision of protein quantification. although better precision of protein quantification can be achieved by a variety of chemical labeling schemes, e.g., SILAC, iTRAQ and TMT labeling.
        </p>
        <p style="font-weight: 500">
            Ref: Marcotte EM. Nat Biotechnol. 2007 Jul;25(7):755-7. PMID: 17621303
            Hu A, et al. F1000Res. 2016; 5: F1000 Faculty Rev-419. PMID: 27092249
        </p>



        <a name="Data-independent-acquisition-(DIA)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Data independent acquisition (DIA)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a process that the fragment ion spectra of all peptides within a mass range are acquired in each cycle time without a pre‐selection of the precursor ions, thus leading to an unbiased recording of all detectable fragment ions derived from all peptide precursors of a given sample. However, this type of data cannot be interpreted by standard search algorithms, since the link between the precursor peptide and its fragment ions are lost. Interpretation of DIA data is also difficult because of the complex MS2 scans derived from the mixtures of peptides. Recent developments in bioinformatics have overcome this DIA issue. The utilization of spectral libraries containing sets of annotated and refined peptide-MS2 spectrum matches from DDA experiments allows comparable breadth of proteome coverage between DIA and DDA. In addition, peptide quantification which relies on comparing DIA spectra to spectral libraries shows highly accurate and reproducible results as SRM/MRM and PRM, although DIA has lower precision in the measuring of very low abundance peptides due to interference of higher abundant co-eluting peptides. See also “SWATH-MS”
        </p>
        <p style="font-weight: 500">
            Ref: Hu A, et al. F1000Res. 2016; 5: F1000 Faculty Rev-419. PMID: 27092249
        </p>



        <a name="Discovery-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Discovery proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Discovery proteomics is the bottom up approach which employs globally unbiased identification and quantification as many proteins as possible in biological samples in the hope of discovering key molecular targets for further study and validation.
        </p>
        <p style="font-weight: 500">
            Ref: Angel TE et al., Chem Soc Rev. 2012 May 21; 41(10): 3912–3928. PMID: 22498958
        </p>



        <a name="Dithiothreitol"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Dithiothreitol</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a reversible reducing agent typically used to break intra- and intermolecular disulfide bonds between cysteine residues of proteins.
        </p>



        <a name="Dwell-time"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Dwell time</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a length of time in which a specific m/z ion signal is collected.
        </p>



        <a name="ECD"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">ECD</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to ECD is based on the interaction of a free electron with a multiply protonated molecule to induce specific N–Cα bond cleavages of the peptide backbone. As compared to CID, ECD provides an extensive fragmentation resulting in improved sequence coverage. Moreover, the nonergodic feature of ECD preserves the labile post-translational modification and consequently enables the generation of ions diagnostic for the modification site(s). The shortcoming of ECD resides in its availability merely on costly and sophisticated high-resolution Fourier-transform ion cyclotron resonance (FT-ICR) MS.
        </p>
        <p style="font-weight: 500">
            Ref: Sarbu M, et al., Amino Acids. 2014;46(7):1625-34. PMID: 24687149
        </p>



        <a name="ESI"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">ESI</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to electrospray ionization. ESI is the most widely used ionization method in proteomic analysis as an interface between LC and MS/MS. The ionization process is based on the formation of sprayed droplets at the end of the electrospray capillary tip that subjected into an intense electrical field toward the orifice of mass spectrometer. The sprayed droplets become evaporated and shrinkage due to surface-Coulombic forces overcome surface-tension forces, ultimately leading to the unsolvated protonated or deprotonated analyte ions. The sensitivity of ESI is improved by high-concentration low-volume samples, which led to the development of low flow-rate ESI, for example, nanoLC-ESI.
        </p>
        <p style="font-weight: 500">
            Ref: Griffiths WJ, et al., Biochem J. 2001 May 1;355(Pt 3):545-61.
            Wilm M. Mol Cell Proteomics. 2011 Jul;10(7):M111.009407. PMID: 21742801
        </p>



        <a name="ETD"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">ETD</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to electron transfer dissociation, an ECD-like activation technique for use on more affordable ion-trap instruments. While CID is a method of choice for peptide sequencing, ETD is the method of choice for the analysis of post-translational modification. ETD induces specific N–Cα bond cleavages of the peptide backbone with the preservation of the post-translational modifications and generation of product ions that are diagnostic for the modification site(s). In addition, ETD contributed significantly to the development of top-down approaches which enable tandem MS of intact protein ions.
        </p>
        <p style="font-weight: 500">
            Ref: Zhou Y, et al., Curr Pharm Biotechnol. 2011 Oct;12(10):1558-67. PMID: 21542796
            Sarbu M, et al., Amino Acids. 2014;46(7):1625-34. PMID: 24687149
        </p>



        <a name="False-discovery-rate-(FDR)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">False discovery rate (FDR)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            FDR is the metric for global confidence assessment of a large-scale proteomics dataset. In the large-scale high-throughput study when multiple independent statistical hypothesis tests are conducted on peptide spectrum matches (PSM) or hits, single hypothesis significance measures (like p-value) are not sufficient since there will be a collection of hits (each with p<0.05) with a final error of more than 5% globally. To address this issue, FDR is applied to adjust for multiple comparison and control the false positives. In the context of proteomics, FDR is the measure of the false positives (incorrect hits above a certain threshold) presented among the accepted hits obtained by a database search algorithm.
            In mass spectrometric-based proteomics, the dominant strategy of FDR calculation is target-decoy (TD) database search, in which the database search is carried out on the target database (i.e. Swiss-Prot) as well as a decoy database. the decoy database is constructed by shuffling, randomizing or by simply reversing the target database. The TD database search can be performed separately or together (concatenated) and scored using a search algorithm. By using this approach, FDR can be easily calculated from the number of false positives divided by the total hits.
        </p>
        <p style="font-weight: 500">
            Ref: Aggarwal S et al. Methods Mol Biol. 2016;1362:119-28. PMID: 26519173
        </p>



        <a name="Filter-aided-sample-preparation-(FASP)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Filter-aided sample preparation (FASP)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            FASP is an effective strategy of processing protein extracts for bottom-up proteomics. After an initial step of protein solubilization in a detergent-contained lysis buffer, FASP repurposes centrifugal ultrafiltration concentrators for removal of detergents, protein cleavage, and isolation of pure peptide fractions for further analysis by mass spectrometry.
        </p>
        <p style="font-weight: 500">
            Ref: Wiśniewski JR. Methods Enzymol. 2017;585:15-27. PMID: 28109427
        </p>



        <a name="GC"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">GC</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to gas chromatography. GC is used to separate volatile compounds in gas phase.
        </p>



        <a name="GCPInfo"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">GCP</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to global chromatin profiles obtained from a targeted proteomic assay of ~60 probes that monitor combinations of post-translational modifications on histones, representing epigenomic regulations of the cells after various perturbations.

            For more information, please see https://panoramaweb.org/project/LINCS/GCP/begin.view
        </p>



        <a name="HCD"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">HCD</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to higher energy collisional dissociation (or beam-type CID) which is available in Orbitrap. Compared with traditional ion trap-based collision-induced dissociation, HCD fragmentation with Orbitrap detection has no low-mass cutoff, high resolution ion detection, and increased ion fragments resulting in higher quality MS/MS spectra. With higher energy, HCD is enabling a wider range of fragmentation pathways. One disadvantage is that spectral acquisition times are up to two-fold longer because more ions are required for Fourier transform detection in the Orbitrap compared with detection of CID spectra in the ion trap via electron multipliers.
        </p>
        <p style="font-weight: 500">
            Ref: Jedrychowski MP, et al., Mol Cell Proteomics. 2011;10(12):M111.009910. PMID: 21917720
        </p>



        <a name="IAA"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">IAA</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Iodoacetamide, an alkylating agent which commonly used as a part of sample preparation and protein digestion before the MS/MS analysis. IAA covalently binds to the thiol group of cysteine residues, thereby blocking disulfide bond formation of proteins.
        </p>



        <a name="iTRAQ"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">iTRAQ</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to Isobaric Tag for Relative and Absolute Quantitation. The iTRAQ reagent can be used for protein identification and quantitation of up to 8 different samples in a MS/MS analysis.
            For more information, please see “www.sciex.com/products/consumables-and-standards/itraq-reagent"
        </p>



        <a name="Label-free-quantification-(LFQ)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Label-free quantification (LFQ)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            LFQ is mainly performed by two strategies; first, spectral counting; and second, intensity-based quantification which based on area under the curve (AUC) or MS1 signal intensity. LFQ gains popularity in proteomics due to simplicity and low cost. However, accurate and efficient normalization is an important issue in LFQ due to the lack of internal standard.
        </p>
        <p style="font-weight: 500">
            Ref: 	Timms JF et al. Methods Mol Biol. 2010;658:19-45. PMID: 20839096
            Mayne J et al. Anal Chem. 2016;88(1):95-121. PMID: 26558748
        </p>



        <a name="Limit-of-Detection-(LOD)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Limit of Detection (LOD)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the lowest concentration or amount of a protein that can be detected confidently.
        </p>



        <a name="Limit-of-quantification-(LOQ)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Limit of quantification (LOQ)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the lowest concentration or amount of a protein that can be quantified confidently.
        </p>



        <a name="LC"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">LC</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to liquid chromatography. LC separates the analyte in the liquid mobile phase based on their physicochemical interaction with a solid stationary phase. LC can be used for analytical or preparative applications.
        </p>



        <a name="LysC"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">LysC</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            A serine protease with high activity and specificity for the carboxyl side of lysine, but not at arginine residues. Unlike trypsin, LysC can cleave lysines followed by prolines.
        </p>



        <a name="MALDI"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">MALDI</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to matrix assisted laser desorption ionization. In MALDI, the analyte is mixed and co-crystallized with the matrix (usually a small organic compound, e.g., -cyano-4-hydroxycinnamic acid) on a target plate. The ionization is triggered by a laser beam, in which the matrix absorbs the laser energy and then transfers to the analyte molecules. Both matrix and analyte molecules are desorbed into gas phase as ions that can be analyzed by mass spectrometric methods, the most common being time-of-flight (TOF). MALDI-mass spectrometry is applicable to measure a wide range of compounds, for example, peptides and proteins, lipids, and small molecules.
        </p>
        <p style="font-weight: 500">
            Ref: Chaurand P, et al. J Am Soc Mass Spectrom. 1999 Feb;10(2):91-103. PMID: 9926404
        </p>



        <a name="Mass-spectrometry"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Mass spectrometry</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Mass spectrometry in proteomics is an instrument system that measures the mass of ions derived from analyte molecules. Mass spectrometry can form, separate, detect molecular ions based on their mass-to-charge ratio (m/z), and then select and fragment ions of interest for specific structural information (MS/MS spectra). By definition, a mass spectrometer consists of an ion source (commonly MALDI or ESI) that ionized peptides into ions, a mass analyzer that measures the mass-to-charge ratio (m/z) and a detector that registers the number of ions at each m/z value. The mass analyzers are central to the technology. There are three main types of mass analyzers used for proteomic experiments, including trapping type instruments (i.e., quadrupole ion trap (QIT), linear ion trap (LIT), Fourier transform ion cyclotron resonance (FT-ICR), and Orbitrap), quadrupole (Q), and time of flight (TOF) instruments. Mass analyzers can be in combination, usually two or three analyzers within one instrument, in order to take advantages of the strength of all combined analyzers simultaneously, for examples, Q-Trap, QQQ, Q-TOF, TOF–TOF, QQ-LIT. These instruments are called hybrid mass spectrometers, and are highly sensitive and also have a high resolution.
        </p>
        <p style="font-weight: 500">
            Ref: Aebersold R and Mann M, Nature 422:198–207. PMID: 12634793
            Woods AG, et al., Adv Exp Med Biol. 2014;806:1-32. PMID: 24952176
        </p>



        <a name="Missing-values"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Missing values</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Missing values are a common issue in quantitative proteomics. Global proteomics datasets typically are missing 20–50% of the total possible peptide values. There are two common types of missing value:<br/>

            1.	Missing at random (MAR)/ missing completely at random (MCAR): value missing that not directly explained by the nature of the peptide or by its measured intensity<br/>

            2.	Missing not at random (MNAR): peptide value missing due to biological based reason
        </p>
        <p style="font-weight: 500">
            Ref: Webb-Robertson BJ et al. J Proteome Res. 2015;14(5):1993-2001. PMID: 25855118
            Lazar C et al. J. Proteome Res., 2016, 15 (4), pp 1116–1125. PMID: 26906401
        </p>



        <a name="Missing-value-imputation"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Missing value imputation</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a process to handle the missing values from quantitative proteomic experiments. In general, these algorithms can be grouped into three categories: i) imputation by a single-digit replacement (e.g., MinDet and MinProb), ii) imputation based on local structures in datasets (i.e., kNN), and iii) imputation based on global structures in datasets (e.g., SVDImpute). Because of the complex mechanisms of missing data in proteomics, no individual method is a single solution for imputation. Choosing a method adapted to the nature of the missing values is more important than choosing a method itself, regardless of the nature of missing values. For example, MNAR are best imputed by MinDet and MinProb, while MAR/MCAR values are well accounted for by kNN and SVDimpute. In addition, imputation should be performed at the peptide level to avoid suboptimal imputation.
        </p>
        <p style="font-weight: 500">
            Ref: Webb-Robertson BJ et al. J Proteome Res. 2015;14(5):1993-2001. PMID: 25855118
            Lazar C et al. J. Proteome Res., 2016, 15 (4), pp 1116–1125. PMID: 26906401
        </p>



        <a name="Multiple-Reaction-Monitoring-(MRM)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Multiple Reaction Monitoring (MRM)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            MRM is a term which represents multiplexing capability of single reaction monitoring (SRM). MRM is frequently used as a synonym of SRM in the literatures.
            Please see “Single Reaction Monitoring (SRM)” for more details.
        </p>
        <p style="font-weight: 500">
            Ref: Picotti P, et al. Nat Methods. 2012 May 30;9(6):555-66. PMID: 22669653
        </p>



        <a name="MS1-ion"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">MS1 ion</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to precursor ion
        </p>



        <a name="MS2-ion"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">MS2 ion</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to fragment ion
        </p>



        <a name="OpenSWATH"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">OpenSWATH</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            OpenSWATH is a software for automated targeted data extraction from DIA (data-independent acquisition) experiments in mass spectrometry. The main workflow consists of OpenSWATH, PyProphet, TRIC, IPF and TAPIR. OpenSWATH is available as a stand-alone library and is also packaged within the OpenMS C++ library for mass-spectrometry based data analysis.

            For more information, please see http://www.openswath.org/en/latest/index.html
        </p>



        <a name="P100Info"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">P100</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the reduced representation phosphoproteomics dataset generated by using a targeted proteomic assay of 96 phosphopeptide probes that are commonly observed in diverse cell types. Information about amino acid sequences, phosphorylation sites, gene symbol and ID and UniProt ID are shown below:
        </p>
        <img  src="/lincsproteomics/images/n.png"
              class="centered">
        <!--<img border="0" src="/lincsproteomics/images/n.png" width="20%" height="20%"-->
        <!--style="align-content: center; ">-->
        <p class="fh5co-sub-lead">
            For more information, please see https://panoramaweb.org/project/LINCS/P100/begin.view
        </p>



        <a name="Parallel-Reaction-Monitoring-(PRM)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Parallel Reaction Monitoring (PRM)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            PRM is a method of targeted proteomic approach. In similar to those in SRM, the precursor ion is selected and fragmented in PRM. The difference for PRM is that full MS/MS spectra are acquired for each precursor (i.e., the parallel detection of all product ions) in the high-resolution accurate-mass (HR/AM) instrument, whereas in SRM only predefined product ions (i.e., transitions) are monitored by the low-resolution quadrupole mass analyzer.
            See also “Targeted proteomics” for more details.
        </p>
        <p style="font-weight: 500">
            Ref: Shi T et al, Proteomics. 2016 Aug; 16(15-16): 2160–2182. PMID: 27302376
        </p>



        <a name="Peakview"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Peakview</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Peakview software is distributed by Sciex (Concord, Ontario, Canada). Peakview is a standalone, general purpose application for exploring and interpreting qualitative LC/MS data from *.wiff files. The software includes tools for analyzing both nominal and accurate mass spectrometry data from single and multiple samples, as well as tools for elucidating chemical structures. With the add-in MS/MSALL SWATH Acquisition MicroApp, Peakview can perform SWATH analysis. The SWATH Acquisition MicroApp generates extracted ion chromatograms (XIC) of fragment ions from MS/MS spectra of targeted proteins and peptides and integrates the peak areas from the SWATH data files.

            For more information, please see https://sciex.com/products/software/peakview-software
        </p>


        <a name="Post-translational-modification-(PTM)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Post-translational modification (PTM)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>
        <p class="fh5co-sub-lead">
            Residue covalent modifications occurring in the specific protein form involved in an interaction.
        </p>
        <p style="font-weight: 500">
            Ref: Controlled vocabularies, HUPO Proteomics Standard Initiative (last maintenance and update Oct 2015) (www.psidev.info/groups/controlled-vocabularies)
        </p>



        <a name="Phosphoproteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Phosphoproteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a sub-discipline of proteomics that is focused on deriving a comprehensive view of the extent and dynamics of protein phosphorylation. The greatest promise of phosphoproteomics is the rapid analysis of entire phosphorylation-based signaling networks.
        </p>
        <p style="font-weight: 500">
            Ref: Mumby M et al., Genome Biol. 2005; 6(9): 230. PMID: 16168091
        </p>



        <a name="Plasma-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Plasma proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            A comprehensive analysis of plasma proteins in healthy and disease conditions which aims to discover and validate biomarkers of clinical values. Plasma proteomics is one of the most complex human derived proteome which is always affected by high abundance proteins (i.e., albumin and immunoglobulin) and has an issue of wide dynamic range of protein concentration. It is generally accepted that plasma need one or more prefractionation to reduce the plasma protein complexity before proteomic analysis.
        </p>



        <a name="Precursor-m/z"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Precursor m/z</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a mass per charge of a precursor peptide
        </p>



        <a name="Precursor-z"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Precursor z</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the charge state of a precursor peptide, usually 2+ - 4+
        </p>



        <!--<a name="Protein"/>-->
        <!--<div style="background-color:#eef2d7">-->
        <!--<h3 class="onerow" style="padding-bottom: 0px">Protein</h3>-->
        <!--<a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>-->
        <!--</div>-->
        <!---->
        <!--<p class="fh5co-sub-lead">-->
        <!--A functional unit of life. Each protein is encoded from its corresponding gene via the process of transcription and translation. Proteins composed of amino acids linked in a linear manner by peptide bond formation between N-terminal and C-terminal of amino acids and folded into different levels of structure (primary to quaternary).-->
        <!--</p>-->



        <a name="Proteoforms"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Proteoforms</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            All different molecular forms of the protein arising from the same gene with changes due to genetic variation, alternative spliced RNA transcripts or post-translational modifications.
        </p>
        <p style="font-weight: 500">
            Ref: Smith LM, et al., Nat Methods 2013;10(3):186-7. PMID: 23443629
        </p>



        <!--<a name="Proteomics"/>-->
        <!--<div style="background-color:#eef2d7">-->
        <!--<h3 class="onerow" style="padding-bottom: 0px">Proteomics</h3>-->
        <!--<a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>-->
        <!--</div>-->
        <!---->
        <!--<p class="fh5co-sub-lead">-->
        <!--Proteomics is the identities, quantities, structures, and biochemical and cellular functions of all proteins in an organism, organ or organelle, and how these vary in space, time and physiological state. In general, a typical proteomic analysis has 3 main steps including sample preparation, protein identification (with or without quantitation) by mass spectrometry, and data interpretation by protein bioinformatics.-->
        <!--</p>-->
        <!--<p style="font-weight: 500">-->
        <!--Ref: Kenyo, G, et al., Mol. Cell Proteomics 2002,1:763-80. PMID: 12438560-->
        <!--</p>-->



        <a name="Proteotypic-peptides"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Proteotypic peptides</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to peptides that uniquely represent targeted proteins or protein isoforms. Identification of proteotypic peptides of target proteins is a pre-requisite step of targeted proteomics workflow. Lists of proteotypic peptides for any particular proteins can be obtained from own proteomic experiments, predicted by using computational tools or retrieved from public databases, i.e., SRMAtlas.
        </p>
        <p style="font-weight: 500">
            Ref: Keerthikumar S et al. Methods Mol Biol. 2017;1549:101-107. PMID: 27975286
        </p>



        <a name="Q1"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Q1</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the mass filter for particular precursor ions
        </p>



        <a name="Q3"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Q3</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the mass filter for particular fragment ions
        </p>



        <a name="Quantitative-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Quantitative proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            There are two main approaches of quantitative proteomics, including the stable isotope-based or label-free methods. Isotope-based methods incorporate heavy versions of specific molecules into the peptides, either by chemical derivatization (i.e., iTRAQ, TMT) or by metabolic labeling (e.g., SILAC). Label-free methods (such as the spectral counting and label-free quantitation) have less accurate peptide quantification than the isotope-based strategies. However, label-free methods are robust and cost-effective, giving these strategies are increasingly attractive for large-scale proteomic studies.
        </p>
        <p style="font-weight: 500">
            Ref: Cox J and Mann M, Annu Rev Biochem. 2011;80:273-99. PMID: 21548781
        </p>



        <a name="Retention-time-(RT)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Retention time (RT)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a particular time of a specific peptide eluted from a LC run.
        </p>



        <a name="SDS"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">SDS</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to Sodium dodecyl sulphate, an ionic detergent constitutes in lysis buffers used for protein solubilization.
        </p>



        <a name="Sequence-tag-identification"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Sequence tag identification</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            This approach leads to protein identification by combining mass measurement and short amino acid sequence information obtained by tandem mass spectrometry. This information is then used to automatically find the best match in a sequence database. The procedure is repeated with several peptides from the digest, resulting in multiple identifications of the same protein or identification of several proteins from the peptide mixture.
        </p>
        <p style="font-weight: 500">
            Ref: Controlled vocabularies, HUPO Proteomics Standard Initiative (last maintenance and update Oct 2015) (www.psidev.info/groups/controlled-vocabularies)
        </p>



        <a name="Shotgun-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Shotgun proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Shotgun proteomics is the high-throughput protein identification in peptide mixture obtained by the proteolytic digestion of complex samples. Shotgun proteomics is the most widely used and the most generic approach of mass spectrometry based proteomics.
            For detail information, please see “Bottom-up proteomics”, “Discovery proteomics” and “Data dependent acquisition”.
        </p>
        <p style="font-weight: 500">
            Ref: Aebersold R and Mann M, Nature 422:198–207. PMID: 12634793
        </p>



        <a name="Skyline"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Skyline</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Skyline is a freely-available, open-source Windows client application for building SRM/MRM, PRM, DIA/SWATH and targeted DDA quantitative methods and analyzing the resulting mass spectrometer data.

            For more information, please see https://skyline.ms/project/home/software/Skyline/begin.view
        </p>



        <a name="SIL-internal-standards"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">SIL internal standards</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to stable isotope labelled analog of the analyte. In the SIL internal standards, several atoms in the analyte are replaced by their stable isotopes, such as deuterium (D), 13C, 15N, or 18O. SIL peptides are typically used as the internal standards to generate the calibration curve for precise and accurate quantitation of targeted proteins in mass spectrometry proteomic assay.
        </p>



        <a name="SILAC"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">SILAC</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to stable isotope labeling using amino acids in cell culture. SILAC is a metabolic labeling method that incorporates “heavy” 13C- or 15N-labeled amino acids into cellular proteins in vivo for quantitative proteomics.

            For more information, please see “https://www.thermofisher.com/us/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/silac-metabolic-labeling-systems.html”
        </p>



        <a name="Single-Reaction-Monitoring (SRM)"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Single Reaction Monitoring (SRM)</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            SRM is a targeted proteomic approach. SRM primarily performs on triple quadrupole (QqQ) mass spectrometry and serves as the MS “gold-standard” for targeted proteomics. SRM is a very sensitive and highly selective method to detect and quantify a target peptide, and by inference, a target protein in complex samples. When applying multiple targets, a term of multiple reaction monitoring (MRM) is used to represent multiplexing capability of SRM.
        </p>
        <h2 style="text-decoration: underline;">Technique:</h2>
        <p class="fh5co-sub-lead">
            SRM utilizes a unique capability of the Triple Quadrapole (QQQ) mass spectrometer to act as mass filters (Q1) which then isolate the target analyte (precursor ion) from the contaminants. The precursor ion is sent to fragmentation in Q2 by collision-induced dissociation. A specific fragment ion from the precursor ion is selected in Q3 and guided to the detector. The number of specific fragment ions that reach the detector is counted over time, resulting in a chromatographic trace with retention time and signal intensity as coordinates. Monitoring several precursor-fragment ion pairs (or transitions) yields the chromatographic peaks at a specific retention time that supports relative or absolute (if an appropriate SIS-peptide is used) quantification of a targeted peptide. A set of suitable transitions can be applied to generate a specific assay to detect and quantify the target peptide, and by inference, the target protein in complex samples.
        </p>
        <h2 style="text-decoration: underline;">Applications:</h2>
        <p class="fh5co-sub-lead">
            1.	Study of protein modifications. For example, SRM/MRM can be used to measure abundance changes of phosphorylated proteins to a series of pertubations in multiple cancer cell lines which can give insights into functional changes in the protein pathways and networks responsible to the pertubagens (P100 set in the iLINC project).
        </p>
        <p class="fh5co-sub-lead">
            2.	Biology and diseases. SRM/MRM can be used to accurately quantify a series of 50-100 proteins involved in cellular biological pathways at different functional/pathological states.
        </p>
        <p class="fh5co-sub-lead">
            3.	Clinical application. SRM/MRM is well-suited for biomarker validation of multiple target proteins across large sample sets of clinical relevant specimens, i.e., plasma, serum and urine. Future developments of SRM/MRM as a platform of multiplexing clinical assays is promising.
        </p>
        <h2 style="text-decoration: underline;">Limitations:</h2>
        <p class="fh5co-sub-lead">
            : Number of measurable proteins is limited to 50-100 proteins per one LC-MS run. Accurate quantification of low-abundant proteins in complex samples is also limited by the sensitivity of SRM measurement, unless sample prefractionation or enrichment steps are applied.
        </p>
        <p style="font-weight: 500">
            Ref: Picotti P, et al. Nat Methods. 2012;9(6):555-66. PMID: 22669653
        </p>



        <a name="SWATH-MS"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">SWATH-MS</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>
        <h3>SWATH-MS</h3>
        <p class="fh5co-sub-lead">
            Refers to Sequential Window Acquisition of all THeoretical fragment ion-Mass Spectrometry. SWATH-MS is a targeted label-free proteomic approach that couples DIA with direct searching of individual samples against the established spectral library. A major advantage is that SWATH-MS can maximize peptide detection at a comparable coverage of shotgun proteomics while still quantitates peptide amount with consistency and reproducibility across various samples as SRM/MRM and PRM, thus increasing experimental efficiency and reducing quantitative variability in the proteomic study.
        </p>
        <img  src="/lincsproteomics/images/7.png"
              class="centered">
        <!--<img border="0" src="/lincsproteomics/images/7.png" width="4%" height="4%"-->
        <!--style="align-content: center; ">-->
        <p class="fh5co-sub-lead">
            (modified from www.sciex.com)
        </p>
        <h2 style="text-decoration: underline;">Technique:</h2>
        <p class="fh5co-sub-lead">
            SWATH-MS is a 2-step process; first, the spectral library generation, and second, SWATH data extraction. The spectral libraries can be simply built using the information of DDA data of samples of interest in which the same set of samples is planned for further analysis in DIA mode. This will give a very basic spectral library for SWATH data extraction. Alternatively, information from multiple DDA runs of technical replicates and/or samples obtained from different genotypes, phenotypes, perturbations and fractionation strategies will help increase the depth of proteome coverage and more inclusive spectral libraries.
            For SWATH data extraction, the DIA data are searched against the generated spectral library that allows a set number of fragment ion chromatograms to be extracted for a peptide within the window of its predicted retention time. The peak groups are scored to discriminate a “true” peptide target from nonspecific noise and distribution of these target scores is modeled against the distribution of scores attributed to decoy peak groups to determine a score cut off, resulting in an acceptable false discovery rate using the target-decoy approach. Peptide abundance is then inferred from the aggregate of the area under the curve for each fragment ion extracted ion chromatograms (XICs), which can be used to estimate the overall protein intensity.
        </p>
        <h2 style="text-decoration: underline;">Applications:</h2>
        <p class="fh5co-sub-lead">
            SWATH-MS is a suitable platform for any proteomic project that need to accurately quantify 1000s peptides across a large number of samples, therefore serving for both discovery and targeted proteomic experiments.
        </p>
        <h2 style="text-decoration: underline;">Limitations:</h2>
        <p class="fh5co-sub-lead">
            SWATH-MS has less selective and sensitive than SRM and PRM. Data analysis always needs sophisticate analytical pipeline and particular software as compared to <a href="" ng-click="scrollTo('Data-dependent-acquisition-(DDA)')">DDA</a>, SRM/MRM or PRM.
        </p>
        <p style="font-weight: 500">
            Ref: Holewinski RJ et al., Methods Mol Biol. 2016;1410:265-79. PMID: 26867750
        </p>



        <a name="Targeted-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Targeted proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to a proteomic approach which specifies targets in complex protein mixture and determines their presence and quantity across a sample cohort. There are three main methods including SRM/MRM, PRM, and SWATH.
        </p>
        <img  src="/lincsproteomics/images/8.png"
              class="centered">
        <!--<img border="0" src="/lincsproteomics/images/8.png" width="20%" height="20%"-->
        <!--style="align-content: center; ">-->
        <p style="font-weight: 500">
            Ref: Vidova V et al. Anal Chim Acta. 2017 Apr 29;964:7-23. PMID: 28351641
            Shi T et al, Proteomics. 2016 Aug; 16(15-16): 2160–2182. PMID: 27302376
        </p>



        <a name="TCEP"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">TCEP</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to tris(2-carboxyethyl)phosphine, an irreversible reducing agent that can use to break disulfide bonds of proteins.
        </p>



        <a name="TMT"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">TMT</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to Tandem Mass Tag reagent. The TMT reagent can be used to enable protein identification and quantitation of up to 10 different samples at a single MS/MS experiment.

            For more information, please see “www.thermofisher.com/us/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/tandem-mass-tag-systems.html”
        </p>



        <a name="Top-down-proteomics"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Top-down proteomics</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the study of the proteoforms - all molecular forms in which the protein products of a single gene can be found including all forms of genetic variation, alternative splicing of RNA transcripts and post-translational modifications. As proteoforms are tightly linked the functions of cells and tissues underlie the complex phenotypes, top-down proteomics will provide critical insights into fundamental process of biological systems.
        </p>
        <img  src="/lincsproteomics/images/9.png"
              class="centered">
        <!--<img border="0" src="/lincsproteomics/images/9.png" width="20%" height="20%"-->
        <!--style="align-content: center; ">-->
        <p style="font-weight: 500">
            Ref: Toby TK, et al., Annu Rev Anal Chem. 2016;9(1):499–519. PMID: 27306313
            Smith LM et al. Science. 2018;359(6380)1106-1107. PMID: 29590032
        </p>



        <a name="Transitions"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Transitions</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            Refers to the specific pair of m/z values associated to the precursor and the selected fragment ions. Transition can be written as parent m/z > fragment m/z (e.g. 440.7 > 201.1).
        </p>
        <p style="font-weight: 500">
            Source: http://www.mrmatlas.org/glossary2.php
        </p>



        <a name="Trypsin"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Trypsin</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            The most common proteolytic enzyme uses in bottom up proteomic experiments. This enzyme cleaves at C-terminus of lysine or arginine, producing peptides from a protein. Note that trypsin cannot cleave lysines follow by prolines, resulting in miss cleavage peptides.
        </p>



        <a name="Ubiquitin-modification"/>
        <div style="background-color:#eef2d7">
            <h3 class="onerow" style="padding-bottom: 0px">Ubiquitin modification</h3>
            <a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>
        </div>

        <p class="fh5co-sub-lead">
            A post-translational modification involving a sequential reaction steps of E1/E2/E3 enzymes leading to ubiquitin activation and transferring to the substrate proteins. Ubiquitin modification (or ubiquitination) starts by the attachment of the 76-amino acid protein ubiquitin to a substrate lysine residue via an isopeptide bond, resulting in ubiquitinated lysine. Ubiquitin modification can occur in mono-ubiquitination and poly-ubiquitination, while poly-ubiquitination can be homotypic and heterotypic linkage types.
            After tryptic digestion, the ubiquitin is cleaved as same as the substrate protein by the enzyme trypsin, resulting in a diglycine remnant of ubiquitin remained on the lysine residue (GG--K) of the substrate peptide. The GG--K is a basis of ubiquitin modified peptide enrichment and bottom-up proteomic analysis.
        </p>
        <p style="font-weight: 500">
            Ref: Swatek KN et. al. Cell Res. 2016;26(4):399-422. PMID: 27012465
        </p>



    </div>
</div>

</body>
</html>