<div id="fh5co-page" ng-init="initNetwork()">


    <section id="fh5co-header">
        <div class="container">
            <nav role="navigation">
                <ul class="pull-left left-menu">
                    <!--	<li><a href="about.html">About</a></li> -->
                    <li><a href="/lincsproteomics/explore">Explore</a></li>

                    <li><a href="/lincsproteomics/download">Download</a></li>
                </ul>
                <h1 id="fh5co-logo"><a href="/lincsproteomics/lincsproteomics">LINCS PROTEOMICS<span>.</span></a></h1>

                <ul class="pull-right right-menu">
                    <li><a href="/lincsproteomics/assays">Assays</a></li>
                    <li class="active"><a href="/lincsproteomics/about">About</a></li>

                </ul>
            </nav>
        </div>
    </section>
    <!--This is for showing header when scroll down-->
    <div class="topMenu">
        <section id="fh5co-header2">
            <div class="container">
                <nav role="navigation">
                    <ul class="pull-left left-menu">
                        <!--	<li><a href="about.html">About</a></li> -->
                        <li><a href="/lincsproteomics/explore">Explore</a></li>
                        <li><a href="/lincsproteomics/download">Download</a></li>
                    </ul>
                    <h3 id="fh5co-logo2"><a href="/lincsproteomics/lincsproteomics">LINCS PROTEOMICS<span>.</span></a>
                    </h3>
                    <ul class="pull-right right-menu">
                        <li><a href="/lincsproteomics/assays">Assays</a></li>
                        <li class="active"><a href="/lincsproteomics/about">About</a></li>

                    </ul>
                </nav>
            </div>
        </section>
    </div>


    <!--
            <section id="fh5co-header">
                <div class="container">
                    <nav role="navigation">
                        <ul class="pull-left left-menu">
                            <li class="active"><a href="about.html">About</a></li>
                            <li><a href="tour.html">Tour</a></li>
                            <li><a href="pricing.html">Pricing</a></li>
                        </ul>
                        <h1 id="fh5co-logo"><a hrindex3.htmlhtml">guide<span>.</span></a></h1>
                        <ul class="pull-right right-menu">
                            <li><a href="#">Login</a></li>
                            <li class="fh5co-cta-btn"><a href="#">Sign up</a></li>
                        </ul>
                    </nav>
                </div>
            </section>-->
    <!-- #fh5co-header -->


    <section id="fh5co-hero" class="no-js-fullheight"
             style="background-image: url(/lincsproteomics/images/University-of-Cincinnati-903.jpg);" data-next="yes">
        <div class="fh5co-overlay"></div>
        <div class="container">
            <div class="fh5co-intro no-js-fullheight">
                <div class="fh5co-intro-text">
                    <!--
                        INFO:
                        Change the class to 'fh5co-right-position' or 'fh5co-center-position' to change the layout position
                        Example:
                        <div class="fh5co-right-position">
                    -->
                    <div class="fh5co-center-position">
                        <!--<h2 class="animate-box" style="font-size: 45px">About Us</h2>-->
                        <h3 class="animate-box">
                            LINCS Proteomics Data Portal enables exploration of LINCS proteomic
                            data and related resources. The portal provides tailored “data-centric”
                            and “assay (or annotation)-centric” views to facilitate navigation, search,
                            and interpretation of LINCS proteomics data.

                        </h3>

                    </div>
                </div>
            </div>
        </div>
        <div class="fh5co-learn-more animate-box">
            <a href="#" class="scroll-btn">
                <span class="text">Explore more about LINCS</span>
                <span class="arrow"><i class="icon-chevron-down"></i></span>
            </a>
        </div>
    </section>
    <!-- END #fh5co-hero -->

    <!--<section id="fh5co-features-3" class="boredr-top" style="padding: 60px 5px 5px 5px">-->
    <!--&lt;!&ndash;<div class="container" >&ndash;&gt;-->
    <!--<div class="row " style="padding: 5px 5px 5px 5px">-->
    <!--<div class="col-md-6 col-md-offset-3 text-center">-->
    <!--<h2 class="fh5co-lead animate-box">What is LINCS?</h2>-->
    <!--<p class="fh5co-sub-lead animate-box">The Library of Network-Based Cellular Signatures-->
    <!--(<a href="http://www.lincsproject.org/" target="_blank">LINCS</a>)-->
    <!--Program aims to create a network-based understanding of biology by cataloging changes in-->
    <!--gene-->
    <!--expression and other cellular processes that occur when cells are exposed to a variety of-->
    <!--perturbing agents. </p>-->
    <!--</div>-->
    <!--</div>-->

    <!---->
    <!--</section>-->

    <div id="fh5co-about">


        <div class="biggerTab">

            <div id="aboutTab">
                <ul class="nav nav-tabs nav-tabs2">
                    <!--<ul class="nav nav-pills nav-stacked col-md-3">-->
                    <li class="active" style="width:25%; ">
                        <a data-toggle="tab" data-target="#assays"
                           style="font-size: 14px; font-weight: 500" ng-click="showAssaysSubTab()">Assays & Glossary</a>
                    </li>
                    <li style="width:25%;">
                        <a data-toggle="tab" data-target="#centers"
                           style="font-size: 14px; font-weight: 500" ng-click="showCentersSubTab()">Centers</a>
                    </li>

                    <li style="width:25%;">
                        <a data-toggle="tab" data-target="#publication"
                           style="font-size: 14px; font-weight: 500" ng-click="showPublicationSubTab()">Publications</a>
                    </li>
                    <li  style="width:25%;">
                        <a data-toggle="tab" data-target="#about"
                           style="font-size: 14px; font-weight: 500" ng-click="showAboutSubTab()">About</a>
                    </li>


                </ul>

                <div class="tab-content tab-content2  text-left"
                     style="padding-top: 20px; padding-bottom:20px; background-color: white;">
                    <!--<div class="tab-content col-md-9 text-left" style="padding-top: 20px">-->
                    <!--<div class="tab-content tab-content2" style="background-color: white; ">-->
                    <div class="tab-pane fade in active"  id="assays">
                        <a name="top"/>
                        <h4 style="align-content: center">Access and
                            <a href="download.html" class="a green">Download</a> Data Sets, Or Explore <a
                                    href="http://lincsportal.ccs.miami.edu/dcic-portal/"
                                    target="_blank"
                                    class="a green">LINCS Data Portal</a> <a
                                    ng-href="http://lincsportal.ccs.miami.edu/dcic-portal/"
                                    target="_blank">
                                <img border="0" src="/lincsproteomics/images/ldp_logo.png" width="4%" height="4%"
                                     style="align-content: center; ">
                            </a>
                        </h4>

                        <!--<div class="row ">-->


                        <!--<div class="bs-example"  append-source>-->
                        <!--<div class="col-md-6" style="float: left;">-->
                        <!--<div class="form-group">-->
                        <!--<h4>Access and-->
                        <!--<a href="download.html" class="a green">Download</a> Data Sets</h4>-->

                        <!--</div>-->


                        <!--</div>-->
                        <!--<div align="right" class="col-md-6">-->
                        <!--<h4>Or Explore <a href="http://lincsportal.ccs.miami.edu/dcic-portal/"-->
                        <!--target="_blank"-->
                        <!--class="a green">LINCS Data Portal</a> <a-->
                        <!--ng-href="http://lincsportal.ccs.miami.edu/dcic-portal/"-->
                        <!--target="_blank">-->
                        <!--<img border="0" src="/lincsproteomics/images/ldp_logo.png" width="4%" height="4%"-->
                        <!--style="align-content: center; ">-->
                        <!--</a></h4>-->


                        <!--</div>-->

                        <!--</div>-->
                        <!--<hr/>-->


                        <!--</div>-->

                        <div style="padding-top: 5px;">
                            <ul class="nav nav-tabs nav-tabs3">
                                <!--<ul class="nav nav-pills nav-stacked col-md-4">-->
                                <!--<li class="active" style="width:25%;">-->
                                <!--<a data-toggle="tab" data-target="#about"-->
                                <!--style="font-size: 14px; font-weight: 500" ng-click="showAboutSubTab()">About</a>-->
                                <!--</li>-->
                                <li class="active" style="width:16.6666667%;"><a data-toggle="tab" href="#P100"
                                                                                 ng-click="showNetwork('P100')">P100
                                    (PCCSE)</a>
                                </li>
                                <li style="width:16.6666667%;"><a data-toggle="tab" href="#GCP"
                                                                  ng-click="showNetwork('GCP')">GCP
                                    (PCCSE)</a></li>
                                <li style="width:16.6666667%;"><a data-toggle="tab" href="#DToxS"
                                                                  ng-click="showNetwork('DToxS')">MS
                                    Protein State (DToxS)</a></li>

                                <li style="width:16.6666667%;"><a data-toggle="tab" href="#SWATH"
                                                                  ng-click="showNetwork('SWATH')">SWATH-MS (NeuroLINCS)</a>

                                <li style="width:16.6666667%;"><a data-toggle="tab" href="#Micro"
                                                                  ng-click="showNetwork('Micro')">Microwestern (DToxS)</a></li>
                                <li style="width:16.65%;"><a data-toggle="tab" href="#RPPA"
                                                             ng-click="showNetwork('RPPA')">RPPA (HMS
                                    LINCS)</a></li>

                            </ul>
                            <div class="tab-content " style="padding-top: 20px">
                                <!--<div class="tab-content col-md-8 text-left" style="padding-top: 20px">-->
                                <!--<div class=tab-content>-->
                                <!--<div class=tab-pane active id=start>1</div>-->
                                <div class="tab-pane fade in active" id="P100">
                                    <p class="fh5co-sub-lead "><a href="" ng-click="scrollTo('P100Info')">P100</a>
                                        has been designed to measure the phosphorylation state of
                                        cellular
                                        systems, using a set
                                        of about hundred phospho-peptide probes whose abundance levels
                                        are
                                        measured
                                        using <a href="#Mass-spectrometry">mass spectrometry</a>
                                        techniques. <a href="#P100Info">P100</a> can be classified as a <a
                                                href="#Targeted-proteomics">'targeted proteomics'</a>
                                        assay that
                                        provides a snapshot of cellular
                                        signaling cascades that involve phosphorylation of the <a
                                                href="#P100Info">P100</a>
                                        target
                                        sites/peptides.
                                        <a href="https://panoramaweb.org/labkey/project/LINCS/begin.view?"
                                           target="_blank">More
                                            ...</a>
                                    </p>
                                </div>
                                <div id="GCP" class="tab-pane fade">
                                    <!--<h3>Menu 1</h3>-->
                                    <p class="fh5co-sub-lead ">
                                        <a href="#GCPInfo">GCP</a> has been designed to measure the histone
                                        modification state, and thus indirectly the chromatin state of
                                        cellular
                                        systems, using a set of about sixty target modified (acetylated,
                                        methylated
                                        etc.)
                                        histone peptides whose abundance levels are measured
                                        using <a href="#Mass-spectrometry">mass spectrometry</a>.
                                        <a href="https://panoramaweb.org/labkey/project/LINCS/begin.view?"
                                           target="_blank">More
                                            ...</a>
                                    </p>
                                </div>
                                <div id="DToxS" class="tab-pane fade">
                                    <!--<h3>Menu 2</h3>-->
                                    <p class="fh5co-sub-lead ">
                                        <a href="#Mass-spectrometry">mass spectrometry</a> is a sensitive technique
                                        that
                                        measures the mass-to-charge ratio of ions to
                                        identify and quantify molecules in simple and
                                        complex mixtures. Samples are loaded into the mass spectrometer
                                        in
                                        liquid,
                                        gas or
                                        dried form
                                        and then vaporized and ionized by the ion source. The charge
                                        that these
                                        molecules
                                        receive allows
                                        the mass spectrometer to accelerate the ions throughout the
                                        remainder of
                                        the
                                        system.
                                        The ions
                                        encounter electric and/or magnetic fields from mass analyzers,
                                        which
                                        deflect
                                        the
                                        paths of individual
                                        ions based on their mass and charge (m/z). Ions that have
                                        successfully
                                        been
                                        deflected by the mass
                                        analyzers then hit the ion detector that registers the number of
                                        ions at
                                        each m/z
                                        value.
                                    </p>
                                </div>
                                <div id="Micro" class="tab-pane fade">
                                    <!--<h3>Menu 2</h3>-->
                                    <p class="fh5co-sub-lead ">
                                        The microwestern is a meso-scale quantitative western blotting assay using infrared fluorescence. Level 1 data are raw signal (background subtracted) from a single "spot" (band). Level 2 data are Level 1 data divided by a loading control signal (e.g. tubulin). Level 3 data are Level 2 data divided by a biological internal control (e.g. Level 2 data corresponding to serum-starved conditions). Level 4 data are averages across Level 3 replicates.

                                    </p>
                                </div>
                                <div id="SWATH" class="tab-pane fade">
                                    <!--<h3>Menu 3</h3>-->
                                    <p class="fh5co-sub-lead ">
                                        SWATH (Sequential Window Acquisition of all THeoretical
                                        fragment ion spectra) is a new strategy for high
                                        throughput, label-free protein quantification. It generates
                                        global,
                                        quantitative
                                        protein maps using
                                        data-independent acquisition of <a
                                            href="#Collision-induced-dissociation-(CID)">collision-induced
                                        dissociation
                                        (CID)</a>
                                        spectra
                                        of all
                                        precursor ions.
                                        As a data-independent acquisition (also referred to as MS/MSall)
                                        <a href="#SWATH-MS">“SWATH-MS”</a>
                                        has a
                                        greater coverage of
                                        peptide identification compared to classical discovery
                                        approaches.
                                        Using known fingerprints of target peptides comprising precursor
                                        mass,
                                        chromatographic retention
                                        time and <a href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a> <a
                                            href="#Transitions">transitions</a> SWATH protein maps can be
                                        interrogated
                                        for
                                        targeted
                                        quantification of proteins
                                        of interest based on high resolution <a
                                            href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a>-like signatures.
                                        SWATH
                                        acquires
                                        all
                                        <a href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a>
                                        <a href="#Transitions">transitions</a> of all
                                        precursors and thus does not require tedious assay development
                                        and
                                        allows
                                        for a more
                                        dynamic data
                                        interpretation compared to classical <a
                                            href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a> experiments.
                                    </p>
                                </div>
                                <div id="RPPA" class="tab-pane fade">
                                    <!--<h3>Menu 3</h3>-->

                                    <p class="fh5co-sub-lead ">
                                        Reverse phase protein array (RPPA) is a high-throughput
                                        technology that performs protein assays on
                                        thousands of samples simultaneously, including tissue and cell
                                        lysates,
                                        serum,
                                        plasma or other
                                        body fluids that measure the levels of protein expression, as
                                        well as
                                        protein
                                        modifications such
                                        as phosphorylation. Protein lysates are arrayed as microspots on
                                        nitrocellulized
                                        coated glass slides
                                        and probed with highly specific antibodies that have been
                                        validated for
                                        RPPA. When
                                        specific antibodies
                                        are used, RPPA can generate 1,000 times more data points using
                                        10,000
                                        times
                                        less
                                        sample volume than
                                        an ordinary western blot, enabling researchers to monitor
                                        quantitative
                                        proteomic
                                        responses for various
                                        time-scale and input-dose gradients simultaneously.
                                    </p>
                                </div>
                            </div>
                        </div>


                        <div style="padding-top: 15px;">
                            <br/>
                            <hr/>
                            <!--<div class="bs-callout bs-callout-primary" ng-show="ctrl.showDToxSdiv">-->
                            <!--<p class="lead"> Mass spectrometry is a sensitive technique that measures the-->
                            <!--mass-to-charge ratio of ions to identify and quantify molecules in simple and-->
                            <!--complex mixtures. Samples are loaded into the mass spectrometer in liquid, gas or-->
                            <!--dried form and then vaporized and ionized by the ion source. The charge that these-->
                            <!--molecules receive allows the mass spectrometer to accelerate the ions throughout the-->
                            <!--remainder of the system. The ions encounter electric and/or magnetic fields from-->
                            <!--mass analyzers, which deflect the paths of individual ions based on their mass and-->
                            <!--charge (m/z). Ions that have successfully been deflected by the mass analyzers then-->
                            <!--hit the ion detector that registers the number of ions at each m/z value.</p>-->
                            <!---->
                            <!--</div>-->
                            <div class="bs-callout bs-callout-primary" ng-show="ctrl.showDToxSdiv">
                                <!--<h3 >MS Protein State assay (LINCS DToxS)    </h3>-->

                                <p class="fh5co-sub-lead "> Grey nodes represent the query protein set, as captured
                                    and/or
                                    quantified by a proteomic assay. Red nodes are kinases that target
                                    (phosphorylate) the query protein set, whereas green nodes represent
                                    downstream targets of kinases among the query protein set.
                                </p>
                            </div>

                            <!--<div class="bs-callout bs-callout-primary" ng-show="ctrl.showSWATHdiv">-->
                            <!--<p class="lead"> SWATH (sequential window acquisition of all theoretical fragment ion-->
                            <!--spectra) is a new strategy for high throughput, label-free protein quantification.-->
                            <!--It generates global, quantitative protein maps using data-independent acquisition of-->
                            <!--collision-induced dissociation (CID) spectra of all precursor ions. As a-->
                            <!--data-independent acquisition (also referred to as MS/MSall) SWATH-MS has a greater-->
                            <!--coverage of peptide identification compared to classical discovery approaches.-->
                            <!--</p>-->
                            <!---->
                            <!--</div>-->
                            <div class="bs-callout bs-callout-primary" ng-show="ctrl.showSWATHdiv">


                                <p class="fh5co-sub-lead "> Grey nodes represent the query protein set, as captured
                                    and/or
                                    quantified by a proteomic assay. Red nodes are kinases that target
                                    (phosphorylate) the query protein set, whereas green nodes represent
                                    downstream targets of kinases among the query protein set.
                                </p>
                            </div>

                            <!--<div class="bs-callout bs-callout-primary" ng-show="ctrl.showRPPAdiv">-->
                            <!--<p class="lead"> Reverse phase protein array (RPPA) is a high-throughput technology that-->
                            <!--performs protein assays on thousands of samples simultaneously, including tissue and-->
                            <!--cell lysates, serum, plasma or other body fluids that measure the levels of protein-->
                            <!--expression, as well as protein modifications such as phosphorylation. Protein-->
                            <!--lysates are arrayed as microspots on nitrocellulized coated glass slides and probed-->
                            <!--with highly specific antibodies that have been validated for RPPA. When specific-->
                            <!--antibodies are used, RPPA can generate 1,000 times more data points using 10,000-->
                            <!--times less sample volume than an ordinary western blot, enabling researchers to-->
                            <!--monitor quantitative proteomic responses for various time-scale and input-dose-->
                            <!--gradients simultaneously.</p>-->
                            <!--<hr/>-->
                            <!--</div>-->

                            <div class="bs-callout bs-callout-primary" ng-show="ctrl.showRPPAdiv && false">

                                <p class="fh5co-sub-lead "> Grey nodes represent the query protein set, as captured
                                    and/or
                                    quantified by a proteomic assay. Red nodes are kinases that target
                                    (phosphorylate) the query protein set, whereas green nodes represent
                                    downstream targets of kinases among the query protein set.
                                </p>
                            </div>

                            <!--<div class="bs-callout bs-callout-primary" ng-show="ctrl.showP100div">-->
                            <!--<p class="lead">P100 has been designed to measure the phosphorylation state of cellular-->
                            <!--systems, using a set of about hundred pospho-peptide probes whose abundance levels-->
                            <!--are measured using mass spectrometry techniques. P100 can be classified as a-->
                            <!--'targeted-->
                            <!--proteomics' assay that provides a snapshot of cellular signaling cascades that-->
                            <!--involve-->
                            <!--phosphorylation of the P100 target sites/peptides.-->

                            <!--<a href="https://panoramaweb.org/labkey/project/LINCS/begin.view?" target="_blank">More-->
                            <!--...</a>-->
                            <!--</p>-->
                            <!--<hr/>-->
                            <!--</div>-->
                            <div class="bs-callout bs-callout-primary" ng-show="ctrl.showMicrodiv">


                                <!--<p class="fh5co-sub-lead "> Grey nodes represent the query protein set, as captured-->
                                <!--and/or-->
                                <!--quantified by a proteomic assay. Red nodes are kinases that target-->
                                <!--(phosphorylate) the query protein set, whereas green nodes represent-->
                                <!--downstream targets of kinases among the query protein set.-->
                                <!--</p>-->
                            </div>

                            <div class="bs-callout bs-callout-primary" ng-show="ctrl.showP100div">


                                <p class="fh5co-sub-lead "> Grey nodes represent the query protein set, as captured
                                    and/or
                                    quantified by a proteomic assay. Red nodes are kinases that target
                                    (phosphorylate) the query protein set, whereas green nodes represent
                                    downstream targets of kinases among the query protein set.
                                </p>
                            </div>

                            <!--<div class="bs-callout bs-callout-primary" ng-show="ctrl.showGCPdiv">-->
                            <!--<p class="lead">GCP has been designed to measure the histone modification state, and-->
                            <!--thus indirectly the chromatin state of cellular-->
                            <!--systems, using a set of about sixty target modified (acetylated, methylated etc.)-->
                            <!--histone peptides whose abundance levels are measured-->
                            <!--using mass spectrometry. <a-->
                            <!--href="https://panoramaweb.org/labkey/project/LINCS/begin.view?"-->
                            <!--target="_blank">More ...</a>-->
                            <!--</p>-->
                            <!--<hr/>-->
                            <!--</div>-->
                            <div class="bs-callout bs-callout-primary" ng-show="ctrl.showGCPdiv">


                                <p class="fh5co-sub-lead "> Grey nodes represent the query protein set, as captured
                                    and/or
                                    quantified by a proteomic assay. Red nodes are kinases that target
                                    (phosphorylate) the query protein set, whereas green nodes represent
                                    downstream targets of kinases among the query protein set.
                                </p>
                            </div>

                            <div class="category-container" style="padding: 5px 0px 5px 0px">
                                <!--<button>-->
                                <!--<a name="download2" href="#">Download SVG-->
                                <!--</a>-->
                                <!--</button>-->

                                <br/>
                                <div ng-show="ctrl.showGCPdiv || ctrl.showP100div|| ctrl.showDToxSdiv|| ctrl.showSWATHdiv" id="chart"></div>

                            </div>
                        </div>

                        <div style="padding-top: 15px;">
                            <h2 class="fh5co-lead " style="background-color:#d1d8ab">Glossary of related
                                definitions:</h2>


                            <a name="2-D-PAGE"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">

                                <h3 class="onerow" style="padding-bottom: 0px">2-D PAGE (Two-dimensional
                                    polyacrylamide gel electrophoresis)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>

                            </div>

                            <p class="fh5co-sub-lead ">
                                2D-PAGE is a separation method for gel-based proteomics (also known as classical
                                proteomic approach). This method separates complex mixture of proteins based on
                                molecular mass and charge, and can resolve hundreds to thousands intact proteins in
                                a single gel. After 2D-PAGE, protein identification is usually performed by in-gel
                                tryptic digestion and <a href="#Mass-spectrometry">mass spectrometry</a>.
                            </p>
                            <h2 style="text-decoration: underline;">Technique:</h2>
                            <p class="fh5co-sub-lead ">
                                2-D PAGE has two steps of protein separation. First, proteins in a complex sample
                                are separated by their molecular charges (or isoelectric points; pI) in an IPG-strip
                                (a precast gel strip with internal pI gradient) using isoelectric focusing. Second,
                                after isoelectric focusing, the separated proteins are resolved in <a
                                    href="#SDS">SDS</a>-PAGE
                                by their molecular weights. This two-step separation results in a 2-D map of
                                numerous protein spots. Ideally, one protein spot contains one proteoform. For
                                protein identification, protein spots of interest are picked up and subjected to
                                in-gel tryptic digestion. The digested peptides can be analyzed by <a
                                    href="#Mass-spectrometry">mass spectrometry</a> to get their peptide sequences.
                            </p>
                            <!--<img src="paris.jpg" alt="Paris" class="center">-->
                            <img src="/lincsproteomics/images/1.png"
                                 class="centered">
                            <h2 style="text-decoration: underline;">Application:</h2>
                            <p class="fh5co-sub-lead ">
                                2D-PAGE is considered a great tool for detecting proteins with potentially changed
                                <a href="#Post-translational-modification-(PTM)">post-translational
                                    modifications</a>, i.e., phosphorylation and glycosylation. Phosphorylation adds
                                negative charges to a protein, causing a shift of phosphorylated protein to more
                                acidic pI. Glycosylation adds glycan branches to a protein, resulting in a shift of
                                molecular size.
                            </p>
                            <h2 style="text-decoration: underline;">Limitations: </h2>
                            <p class="fh5co-sub-lead ">
                                Although gel-based proteomics has very simple analytical workflow, as a front-end
                                separation strategy, 2-D PAGE is a laborious and time-consuming procedure which are
                                disadvantages for large-scale proteomic projects. One needs a period of training to
                                master this procedure and to achieve an acceptable level of result reproducibility.
                                In addition, depth of proteome coverage from a single analysis is much lower than
                                mass spectrometric-based proteomics. All these limitations lead to decreased
                                popularity of 2-D PAGE and gel-based proteomics in the field.
                            </p>
                            <p style="font-weight: 500">
                                Ref: May C, et al., Methods Mol Biol. 2012;893:37-46. PMID: 22665292
                            </p>


                            <a name="AspN"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">

                                <h3 class="onerow" style="padding-bottom: 0px">AspN</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>

                            </div>

                            <p class="fh5co-sub-lead ">
                                A metalloprotease enzyme that cleaves amino side of aspartic acid residues.
                            </p>


                            <a name="b-and-y-ion-series"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">

                                <h3 class="onerow" style="padding-bottom: 0px">b- and y-ion series</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>

                            </div>

                            <p class="fh5co-sub-lead ">
                                Refers to fragment ions derived from peptide breakage at the peptide bonds during
                                mass spectrometric analysis. The b- and y-ions are indicated if the charge retained
                                on the N-terminus and C-terminus fragments, respectively.
                            </p>
                            <img src="/lincsproteomics/images/2.png" class="centered">
                            <p class="fh5co-sub-lead ">
                                The mass differences obtained from the b- and y-ion series of a peptide can be used
                                to interpret its amino acid sequence.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Marcotte EM. Nat Biotechnol. 2007 Jul;25(7):755-7. PMID: 17621303
                            </p>


                            <a name="CAD"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">

                                <h3 class="onerow" style="padding-bottom: 0px">CAD</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>

                            </div>

                            <p class="fh5co-sub-lead ">
                                Refers to collisional activation dissociation, a synonym of <a
                                    href="#Collision-induced-dissociation-(CID)">CID</a>.
                            </p>


                            <a name="Bottom-up-proteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Bottom-up proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead ">
                                Refers to the characterization of proteins by analysis of proteolytic peptides. Of
                                several proteases, <a href="#Trypsin">trypsin</a> has become a gold standard for
                                protein digestion in bottom-up analysis. When bottom-up proteomics is performed on a
                                mixture of proteins in complex samples for discovery purposes, it is called <a
                                    href="#Shotgun-proteomics">shotgun proteomics</a>. See also <a
                                    href="#Top-down-proteomics">“Top-down proteomics”</a> for comparison.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Zhang Y, et al. Chem Rev. 2013 Apr 10;113(4):2343-94. PMID: 23438204
                            </p>


                            <a name="Carbamidomethylation-(CAM)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Carbamidomethylation (CAM)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead ">
                                CAM is a chemical modification introduced to cysteine residues by alkylation with
                                iodoacetamide during proteomic sample preparation.
                            </p>


                            <a name="Chemical-proteomics-(also-known-as-chemoproteomics)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Chemical proteomics (also known as
                                    chemoproteomics)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead ">
                                Refers to a research area at the interface between chemistry, biochemistry and cell
                                biology that focused on the mechanisms of actions of small molecular compounds i.e.,
                                the mapping of their interacted proteins and their effects on protein expression and
                                <a href="#Post-translational-modification-(PTM)">post-translational modification</a>
                                in target cells or tissue of interest. There are two main approaches; first, global
                                chemoproteomic profiling to evaluate the cellular response to small molecule
                                treatment in a global view; and second, targeted chemoproteomic approach which
                                adopted affinity-based techniques to determine protein binding profiles of small
                                molecular drug
                            </p>
                            <p style="font-weight: 500">
                                Ref: Bantscheff M. Methods Mol Biol. 2012;803:3-13. PMID: 22065214
                            </p>


                            <a name="Clinical-proteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Clinical proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the application of proteomic analysis with the aim of solving a specific
                                clinical problem within the context of a clinical study. The value of clinical
                                proteomics lies in its application with the ultimate goal to improve patient care,
                                for example, detecting biomarkers for early diagnosis, classification for
                                patient-tailored therapy, evaluation of therapeutic response, etc. Accordingly,
                                clinical proteomics should follow the principles and rules of clinical
                                trials/studies, when proteomics should be used to obtain biomarker outcomes.
                                Recommended steps for clinical proteomics are shown as follows:
                            </p>


                            <h3>Recommended steps for clinical proteomics</h3>
                            <p class="fh5co-sub-lead">
                                1) Define a clear clinical question and how the outcome of the study would improve
                                the diagnosis and/or treatment of the disease.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                2) Define the patient and control populations, clinical data to be collected, as
                                well as protocols for sampling and sample preparation.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                3) Define the type of samples needed for the discovery and validation phases.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                4) Define and validate the analytical platforms for discovery (those for validation
                                may well differ).<br/>
                                <!--</p >-->
                                <!--<p class="fh5co-sub-lead">-->
                                5) Obtain IRB approval and written informed consent from the participant.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                6) Perform a pilot study on a validated discovery platform.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                7) Statistically evaluate data from the pilot study to calculate the number of cases
                                and controls for the training set.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                8) Perform study of the training set on the validated platform based on the
                                calculated number of cases and controls.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                9) Evaluate findings from the training set on blinded samples.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                10) Deposit datasets in a public database.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                11) Using these results, transfer the assay to the application platform and test
                                using a training (if applicable) and subsequently a blinded set.<br/>
                                <!--</p>-->
                                <!--<p class="fh5co-sub-lead">-->
                                12) Apply towards clinical use to show whether the findings improve the current
                                clinical situation.<br/>
                            </p>
                            <p style="font-weight: 500">
                                Ref: Mischak H, et al. Proteomics Clin. Appl. 2007, 1, 148–156. PMID: 21136664.<br/>
                            </p>


                            <a name="CE"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">CE</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to capillary electrophoresis, an alternative to chromatography for peptide
                                and intact protein separation prior to mass spectrometric analysis.
                            </p>


                            <a name="Collision-induced-dissociation-(CID)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Collision induced dissociation
                                    (CID)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Dissociation of precursor ions into fragment ions by collision with gas molecules
                                within the collision cell of mass spectrometer.
                            </p>
                            <img src="/lincsproteomics/images/3.png"
                                 class="centered">
                            <!--<img border="0" src="/lincsproteomics/images/3.png" width="4%" height="4%"-->
                            <!--style="align-content: center; ">-->


                            <a name="Cycle-time"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Cycle time</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the sum of the <a href="#Dwell-time">dwell times</a> of all analyte ions
                                in a given set.
                            </p>


                            <a name="Data-dependent-acquisition-(DDA)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Data-dependent acquisition (DDA)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a process driven by an automated data acquisition that trigger MS/MS scan
                                depending on the signal intensity (or abundance) of the precursor (or <a
                                    href="#MS1-ion">MS1</a>) ions. A subset of the most abundant precursor ions
                                reaching the mass spectrometers detector during <a href="#MS1-ion">MS1</a> scan are
                                individually isolated and fragmented in sequential MS/MS scan and can be analyzed
                                with a database search algorithm. Instrument setting up and data analysis of DDA is
                                simple, while this method typically yields 1000s of protein identifications. These
                                advantages make DDA the preferred <a href="#LC">LC</a>-MS/MS method among the wider
                                scientific community. However, DDA method analyzes only a limited number of all
                                precursor ions (identifying the most abundant peptides but missing the rest) and
                                therefore has poor reproducibility of the detection and measurement of low-abundance
                                proteins in complex samples. Run-to-run peptide identification overlap for a given
                                sample is approximately 60%. DDA using a label-free approach has low precision of
                                protein quantification. although better precision of protein quantification can be
                                achieved by a variety of chemical labeling schemes, e.g., <a href="#SILAC">SILAC</a>,
                                <a href="#iTRAQ">iTRAQ</a> and <a href="#TMT">TMT</a> labeling.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Marcotte EM. Nat Biotechnol. 2007 Jul;25(7):755-7. PMID: 17621303
                                Hu A, et al. F1000Res. 2016; 5: F1000 Faculty Rev-419. PMID: 27092249
                            </p>


                            <a name="Data-independent-acquisition-(DIA)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Data independent acquisition
                                    (DIA)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a process that the fragment ion spectra of all peptides within a mass
                                range are acquired in each <a href="#Cycle-time">cycle time</a> without a
                                pre‐selection of the precursor ions, thus leading to an unbiased recording of all
                                detectable fragment ions derived from all peptide precursors of a given sample.
                                However, this type of data cannot be interpreted by standard search algorithms,
                                since the link between the precursor peptide and its fragment ions are lost.
                                Interpretation of DIA data is also difficult because of the complex <a
                                    href="#MS2-ion">MS2</a> scans derived from the mixtures of peptides. Recent
                                developments in bioinformatics have overcome this DIA issue. The utilization of
                                spectral libraries containing sets of annotated and refined peptide-<a
                                    href="#MS2-ion">MS2</a> spectrum matches from <a
                                    href="#Data-dependent-acquisition-(DDA).">DDA</a> experiments allows comparable
                                breadth of proteome coverage between DIA and <a
                                    href="#Data-dependent-acquisition-(DDA).">DDA</a>. In addition, peptide
                                quantification which relies on comparing DIA spectra to spectral libraries shows
                                highly accurate and reproducible results as <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a>/<a
                                    href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a> and <a
                                    href="#Parallel-Reaction-Monitoring-(PRM)">PRM</a>, although DIA has lower
                                precision in the measuring of very low abundance peptides due to interference of
                                higher abundant co-eluting peptides. See also <a href="#SWATH-MS">“SWATH-MS”</a>.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Hu A, et al. F1000Res. 2016; 5: F1000 Faculty Rev-419. PMID: 27092249
                            </p>


                            <a name="Discovery-proteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Discovery proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Discovery proteomics is the bottom up approach which employs globally unbiased
                                identification and quantification as many proteins as possible in biological samples
                                in the hope of discovering key molecular targets for further study and validation.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Angel TE et al., Chem Soc Rev. 2012 May 21; 41(10): 3912–3928. PMID: 22498958
                            </p>


                            <a name="Dithiothreitol"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Dithiothreitol</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a reversible reducing agent typically used to break intra- and
                                intermolecular disulfide bonds between cysteine residues of proteins.
                            </p>


                            <a name="Dwell-time"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Dwell time</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a length of time in which a specific m/z ion signal is collected.
                            </p>


                            <a name="ECD"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">ECD</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to ECD is based on the interaction of a free electron with a multiply
                                protonated molecule to induce specific N–Cα bond cleavages of the peptide backbone.
                                As compared to <a href="#Collision-induced-dissociation-(CID)">CID</a>, ECD provides
                                an extensive fragmentation resulting in improved sequence coverage. Moreover, the
                                nonergodic feature of ECD preserves the labile <a
                                    href="#Post-translational-modification-(PTM)">post-translational
                                modification</a> and consequently enables the generation of ions diagnostic for the
                                modification site(s). The shortcoming of ECD resides in its availability merely on
                                costly and sophisticated high-resolution Fourier-transform ion cyclotron resonance
                                (FT-ICR) MS.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Sarbu M, et al., Amino Acids. 2014;46(7):1625-34. PMID: 24687149
                            </p>


                            <a name="ESI"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">ESI</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to electrospray ionization. ESI is the most widely used ionization method in
                                proteomic analysis as an interface between <a href="#LC">LC</a> and MS/MS. The
                                ionization process is based on the formation of sprayed droplets at the end of the
                                electrospray capillary tip that subjected into an intense electrical field toward
                                the orifice of mass spectrometer. The sprayed droplets become evaporated and
                                shrinkage due to surface-Coulombic forces overcome surface-tension forces,
                                ultimately leading to the unsolvated protonated or deprotonated analyte ions. The
                                sensitivity of ESI is improved by high-concentration low-volume samples, which led
                                to the development of low flow-rate ESI, for example, nanoLC-ESI.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Griffiths WJ, et al., Biochem J. 2001 May 1;355(Pt 3):545-61.
                                Wilm M. Mol Cell Proteomics. 2011 Jul;10(7):M111.009407. PMID: 21742801
                            </p>


                            <a name="ETD"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">ETD</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to electron transfer dissociation, an <a href="#ECD">ECD</a>-like activation
                                technique for use on more affordable ion-trap instruments. While <a
                                    href="#Collision-induced-dissociation-(CID)">CID</a> is a method of choice for
                                peptide sequencing, ETD is the method of choice for the analysis of <a
                                    href="#Post-translational-modification-(PTM)">post-translational
                                modification</a>. ETD induces specific N–Cα bond cleavages of the peptide backbone
                                with the preservation of the <a href="#Post-translational-modification-(PTM)">post-translational
                                modifications</a> and generation of product ions that are diagnostic for the
                                modification site(s). In addition, ETD contributed significantly to the development
                                of top-down approaches which enable tandem MS of intact protein ions.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Zhou Y, et al., Curr Pharm Biotechnol. 2011 Oct;12(10):1558-67. PMID: 21542796
                                Sarbu M, et al., Amino Acids. 2014;46(7):1625-34. PMID: 24687149
                            </p>


                            <a name="False-discovery-rate-(FDR)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">False discovery rate (FDR)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                FDR is the metric for global confidence assessment of a large-scale proteomics
                                dataset. In the large-scale high-throughput study when multiple independent
                                statistical hypothesis tests are conducted on peptide spectrum matches (PSM) or
                                hits, single hypothesis significance measures (like p-value) are not sufficient
                                since there will be a collection of hits (each with p<0.05) with a final error of
                                more than 5% globally. To address this issue, FDR is applied to adjust for multiple
                                comparison and control the false positives. In the context of proteomics, FDR is the
                                measure of the false positives (incorrect hits above a certain threshold) presented
                                among the accepted hits obtained by a database search algorithm.
                                In mass spectrometric-based proteomics, the dominant strategy of FDR calculation is
                                target-decoy (TD) database search, in which the database search is carried out on
                                the target database (i.e. Swiss-Prot) as well as a decoy database. the decoy
                                database is constructed by shuffling, randomizing or by simply reversing the target
                                database. The TD database search can be performed separately or together
                                (concatenated) and scored using a search algorithm. By using this approach, FDR can
                                be easily calculated from the number of false positives divided by the total hits.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Aggarwal S et al. Methods Mol Biol. 2016;1362:119-28. PMID: 26519173
                            </p>


                            <a name="Filter-aided-sample-preparation-(FASP)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Filter-aided sample preparation
                                    (FASP)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                FASP is an effective strategy of processing protein extracts for <a
                                    href="#Bottom-up-proteomics">bottom-up proteomics</a>. After an initial step of
                                protein solubilization in a detergent-contained lysis buffer, FASP repurposes
                                centrifugal ultrafiltration concentrators for removal of detergents, protein
                                cleavage, and isolation of pure peptide fractions for further analysis by <a
                                    href="#Mass-spectrometry">mass spectrometry</a>.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Wiśniewski JR. Methods Enzymol. 2017;585:15-27. PMID: 28109427
                            </p>


                            <a name="GC"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">GC</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to gas chromatography. GC is used to separate volatile compounds in gas
                                phase.
                            </p>


                            <a name="GCPInfo"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">GCP</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to global chromatin profiles obtained from a targeted proteomic assay of ~60
                                probes that monitor combinations of <a
                                    href="#Post-translational-modification-(PTM)">post-translational
                                modifications</a> on histones, representing epigenomic regulations of the cells
                                after various perturbations.

                                For more information, please see
                                https://panoramaweb.org/project/LINCS/GCP/begin.view
                            </p>


                            <a name="HCD"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">HCD</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to higher energy collisional dissociation (or beam-type <a
                                    href="#Collision-induced-dissociation-(CID)">CID</a>) which is available in
                                Orbitrap. Compared with traditional ion trap-based collision-induced dissociation,
                                HCD fragmentation with Orbitrap detection has no low-mass cutoff, high resolution
                                ion detection, and increased ion fragments resulting in higher quality MS/MS
                                spectra. With higher energy, HCD is enabling a wider range of fragmentation
                                pathways. One disadvantage is that spectral acquisition times are up to two-fold
                                longer because more ions are required for Fourier transform detection in the
                                Orbitrap compared with detection of <a
                                    href="#Collision-induced-dissociation-(CID)">CID</a>
                                spectra in the ion trap via electron multipliers.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Jedrychowski MP, et al., Mol Cell Proteomics. 2011;10(12):M111.009910. PMID:
                                21917720
                            </p>


                            <a name="IAA"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">IAA</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Iodoacetamide, an alkylating agent which commonly used as a part of sample
                                preparation and protein digestion before the MS/MS analysis. IAA covalently binds to
                                the thiol group of cysteine residues, thereby blocking disulfide bond formation of
                                proteins.
                            </p>


                            <a name="iTRAQ"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">iTRAQ</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to Isobaric Tag for Relative and Absolute Quantitation. The iTRAQ reagent can
                                be used for protein identification and quantitation of up to 8 different samples in
                                a MS/MS analysis.
                                For more information, please see
                                “www.sciex.com/products/consumables-and-standards/itraq-reagent"
                            </p>


                            <a name="Label-free-quantification-(LFQ)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Label-free quantification (LFQ)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                LFQ is mainly performed by two strategies; first, spectral counting; and second,
                                intensity-based quantification which based on area under the curve (AUC) or <a
                                    href="#MS1-ion">MS1</a> signal intensity. LFQ gains popularity in proteomics due
                                to simplicity and low cost. However, accurate and efficient normalization is an
                                important issue in LFQ due to the lack of internal standard.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Timms JF et al. Methods Mol Biol. 2010;658:19-45. PMID: 20839096
                                Mayne J et al. Anal Chem. 2016;88(1):95-121. PMID: 26558748
                            </p>


                            <a name="Limit-of-Detection-(LOD)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Limit of Detection (LOD)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the lowest concentration or amount of a protein that can be detected
                                confidently.
                            </p>


                            <a name="Limit-of-quantification-(LOQ)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Limit of quantification (LOQ)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the lowest concentration or amount of a protein that can be quantified
                                confidently.
                            </p>


                            <a name="LC"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">LC</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to liquid chromatography. LC separates the analyte in the liquid mobile phase
                                based on their physicochemical interaction with a solid stationary phase. LC can be
                                used for analytical or preparative applications.
                            </p>


                            <a name="LysC"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">LysC</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                A serine protease with high activity and specificity for the carboxyl side of
                                lysine, but not at arginine residues. Unlike <a href="#Trypsin">trypsin</a>, LysC
                                can cleave lysines followed by prolines.
                            </p>


                            <a name="MALDI"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">MALDI</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to matrix assisted laser desorption ionization. In MALDI, the analyte is
                                mixed and co-crystallized with the matrix (usually a small organic compound, e.g.,
                                -cyano-4-hydroxycinnamic acid) on a target plate. The ionization is triggered by a
                                laser beam, in which the matrix absorbs the laser energy and then transfers to the
                                analyte molecules. Both matrix and analyte molecules are desorbed into gas phase as
                                ions that can be analyzed by mass spectrometric methods, the most common being
                                time-of-flight (TOF). MALDI-<a href="#Mass-spectrometry">mass spectrometry</a> is
                                applicable to measure a wide range of compounds, for example, peptides and proteins,
                                lipids, and small molecules.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Chaurand P, et al. J Am Soc Mass Spectrom. 1999 Feb;10(2):91-103. PMID: 9926404
                            </p>


                            <a name="Mass-spectrometry"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Mass spectrometry</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Mass spectrometry in proteomics is an instrument system that measures the mass of
                                ions derived from analyte molecules. Mass spectrometry can form, separate, detect
                                molecular ions based on their mass-to-charge ratio (m/z), and then select and
                                fragment ions of interest for specific structural information (MS/MS spectra). By
                                definition, a mass spectrometer consists of an ion source (commonly <a
                                    href="#MALDI">MALDI</a> or <a href="#ESI">ESI</a>) that ionized peptides into
                                ions, a mass analyzer that measures the mass-to-charge ratio (m/z) and a detector
                                that registers the number of ions at each m/z value. The mass analyzers are central
                                to the technology. There are three main types of mass analyzers used for proteomic
                                experiments, including trapping type instruments (i.e., quadrupole ion trap (QIT),
                                linear ion trap (LIT), Fourier transform ion cyclotron resonance (FT-ICR), and
                                Orbitrap), quadrupole (Q), and time of flight (TOF) instruments. Mass analyzers can
                                be in combination, usually two or three analyzers within one instrument, in order to
                                take advantages of the strength of all combined analyzers simultaneously, for
                                examples, Q-Trap, QQQ, Q-TOF, TOF–TOF, QQ-LIT. These instruments are called hybrid
                                mass spectrometers, and are highly sensitive and also have a high resolution.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Aebersold R and Mann M, Nature 422:198–207. PMID: 12634793
                                Woods AG, et al., Adv Exp Med Biol. 2014;806:1-32. PMID: 24952176
                            </p>


                            <a name="Missing-values"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Missing values</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Missing values are a common issue in quantitative proteomics. Global proteomics
                                datasets typically are missing 20–50% of the total possible peptide values. There
                                are two common types of missing value:<br/>

                                1. Missing at random (MAR)/ missing completely at random (MCAR): value missing that
                                not directly explained by the nature of the peptide or by its measured
                                intensity<br/>

                                2. Missing not at random (MNAR): peptide value missing due to biological based
                                reason
                            </p>
                            <p style="font-weight: 500">
                                Ref: Webb-Robertson BJ et al. J Proteome Res. 2015;14(5):1993-2001. PMID: 25855118
                                Lazar C et al. J. Proteome Res., 2016, 15 (4), pp 1116–1125. PMID: 26906401
                            </p>


                            <a name="Missing-value-imputation"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Missing value imputation</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a process to handle the <a href="#Missing-values">missing values</a> from
                                quantitative proteomic experiments. In general, these algorithms can be grouped into
                                three categories: i) imputation by a single-digit replacement (e.g., MinDet and
                                MinProb), ii) imputation based on local structures in datasets (i.e., kNN), and iii)
                                imputation based on global structures in datasets (e.g., SVDImpute). Because of the
                                complex mechanisms of missing data in proteomics, no individual method is a single
                                solution for imputation. Choosing a method adapted to the nature of the <a
                                    href="#Missing-values">missing values</a> is more important than choosing a
                                method itself, regardless of the nature of <a href="#Missing-values">missing
                                values</a>. For example, MNAR are best imputed by MinDet and MinProb, while MAR/MCAR
                                values are well accounted for by kNN and SVDimpute. In addition, imputation should
                                be performed at the peptide level to avoid suboptimal imputation.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Webb-Robertson BJ et al. J Proteome Res. 2015;14(5):1993-2001. PMID: 25855118
                                Lazar C et al. J. Proteome Res., 2016, 15 (4), pp 1116–1125. PMID: 26906401
                            </p>


                            <a name="Multiple-Reaction-Monitoring-(MRM)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Multiple Reaction Monitoring
                                    (MRM)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                MRM is a term which represents multiplexing capability of <a
                                    href="#Single-Reaction-Monitoring (SRM)">single reaction monitoring (SRM)</a>.
                                MRM is frequently used as a synonym of <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a>
                                in the literatures.
                                Please see <a href="#Single-Reaction-Monitoring (SRM)">"Single Reaction Monitoring
                                (SRM)”</a> for more details.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Picotti P, et al. Nat Methods. 2012 May 30;9(6):555-66. PMID: 22669653
                            </p>


                            <a name="MS1-ion"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">MS1 ion</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to precursor ion
                            </p>


                            <a name="MS2-ion"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">MS2 ion</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to fragment ion
                            </p>


                            <a name="OpenSWATH"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">OpenSWATH</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                OpenSWATH is a software for automated targeted data extraction from <a
                                    href="#Data-independent-acquisition-(DIA)">DIA (data-independent
                                acquisition)</a> experiments in <a href="#Mass-spectrometry">mass spectrometry</a>.
                                The main workflow consists of OpenSWATH, PyProphet, TRIC, IPF and TAPIR. OpenSWATH
                                is available as a stand-alone library and is also packaged within the OpenMS C++
                                library for mass-spectrometry based data analysis.

                                For more information, please see http://www.openswath.org/en/latest/index.html
                            </p>


                            <a name="P100Info"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">P100</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the reduced representation <a
                                    href="#Phosphoproteomics">phosphoproteomics</a> dataset generated by using a
                                targeted proteomic assay of 96 phosphopeptide probes that are commonly observed in
                                diverse cell types.
                                <!--Information about amino acid sequences, phosphorylation sites,-->
                                <!--gene symbol and ID and UniProt ID are shown below:-->
                            </p>
                            <!--<img src="/lincsproteomics/images/n.png"-->
                            <!--class="centered">-->
                            <!--<img border="0" src="/lincsproteomics/images/n.png" width="20%" height="20%"-->
                            <!--style="align-content: center; ">-->
                            <p class="fh5co-sub-lead">
                                For more information, please see
                                https://panoramaweb.org/project/LINCS/P100/begin.view
                            </p>


                            <a name="Parallel-Reaction-Monitoring-(PRM)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Parallel Reaction Monitoring
                                    (PRM)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                PRM is a method of targeted proteomic approach. In similar to those in <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a>, the precursor ion is selected
                                and fragmented in PRM. The difference for PRM is that full MS/MS spectra are
                                acquired for each precursor (i.e., the parallel detection of all product ions) in
                                the high-resolution accurate-mass (HR/AM) instrument, whereas in <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a> only predefined product ions
                                (i.e., <a href="#Transitions">transitions</a>) are monitored by the low-resolution
                                quadrupole mass analyzer.
                                See also <a href="#Targeted-proteomics">"targeted proteomics"</a> for more details.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Shi T et al, Proteomics. 2016 Aug; 16(15-16): 2160–2182. PMID: 27302376
                            </p>


                            <a name="Peakview"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Peakview</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Peakview software is distributed by Sciex (Concord, Ontario, Canada). Peakview is a
                                standalone, general purpose application for exploring and interpreting qualitative
                                <a href="#LC">LC</a>/MS data from *.wiff files. The software includes tools for
                                analyzing both nominal and accurate <a href="#Mass-spectrometry">mass
                                spectrometry</a> data from single and multiple samples, as well as tools for
                                elucidating chemical structures. With the add-in MS/MSALL <a href="#SWATH-MS">SWATH</a>
                                Acquisition MicroApp, Peakview can perform <a href="#SWATH-MS">SWATH</a> analysis. The
                                <a href="#SWATH-MS">SWATH</a> Acquisition MicroApp generates extracted ion
                                chromatograms (XIC) of fragment ions from MS/MS spectra of targeted proteins and
                                peptides and integrates the peak areas from the <a href="#SWATH-MS">SWATH</a> data
                                files.

                                For more information, please see
                                https://sciex.com/products/software/peakview-software
                            </p>


                            <a name="Post-translational-modification-(PTM)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Post-translational modification
                                    (PTM)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>
                            <p class="fh5co-sub-lead">
                                Residue covalent modifications occurring in the specific protein form involved in an
                                interaction.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Controlled vocabularies, HUPO Proteomics Standard Initiative (last maintenance
                                and update Oct 2015) (www.psidev.info/groups/controlled-vocabularies)
                            </p>


                            <a name="Phosphoproteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Phosphoproteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a sub-discipline of proteomics that is focused on deriving a comprehensive
                                view of the extent and dynamics of protein phosphorylation. The greatest promise of
                                phosphoproteomics is the rapid analysis of entire phosphorylation-based signaling
                                networks.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Mumby M et al., Genome Biol. 2005; 6(9): 230. PMID: 16168091
                            </p>


                            <a name="Plasma-proteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Plasma proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                A comprehensive analysis of plasma proteins in healthy and disease conditions which
                                aims to discover and validate biomarkers of clinical values. Plasma proteomics is
                                one of the most complex human derived proteome which is always affected by high
                                abundance proteins (i.e., albumin and immunoglobulin) and has an issue of wide
                                dynamic range of protein concentration. It is generally accepted that plasma need
                                one or more prefractionation to reduce the plasma protein complexity before
                                proteomic analysis.
                            </p>


                            <a name="Precursor-m/z"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Precursor m/z</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a mass per charge of a precursor peptide
                            </p>


                            <a name="Precursor-z"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Precursor z</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the charge state of a precursor peptide, usually 2+ - 4+
                            </p>


                            <!--<a name="Protein"/>-->
                            <!--<div style="background-color:#eef2d7;padding-top: 60px">-->
                            <!--<h3 class="onerow" style="padding-bottom: 0px">Protein</h3>-->
                            <!--<a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>-->
                            <!--</div>-->
                            <!---->
                            <!--<p class="fh5co-sub-lead">-->
                            <!--A functional unit of life. Each protein is encoded from its corresponding gene via the process of transcription and translation. Proteins composed of amino acids linked in a linear manner by peptide bond formation between N-terminal and C-terminal of amino acids and folded into different levels of structure (primary to quaternary).-->
                            <!--</p>-->


                            <a name="Proteoforms"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Proteoforms</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                All different molecular forms of the protein arising from the same gene with changes
                                due to genetic variation, alternative spliced RNA transcripts or <a
                                    href="#Post-translational-modification-(PTM)">post-translational
                                modifications</a>.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Smith LM, et al., Nat Methods 2013;10(3):186-7. PMID: 23443629
                            </p>


                            <!--<a name="Proteomics"/>-->
                            <!--<div style="background-color:#eef2d7;padding-top: 60px">-->
                            <!--<h3 class="onerow" style="padding-bottom: 0px">Proteomics</h3>-->
                            <!--<a class="onerow" style="float:right; padding-bottom: 0px" href="" ng-click="scrollTo('top')">Back to top</a>-->
                            <!--</div>-->
                            <!---->
                            <!--<p class="fh5co-sub-lead">-->
                            <!--Proteomics is the identities, quantities, structures, and biochemical and cellular functions of all proteins in an organism, organ or organelle, and how these vary in space, time and physiological state. In general, a typical proteomic analysis has 3 main steps including sample preparation, protein identification (with or without quantitation) by <a href="#Mass-spectrometry">mass spectrometry</a>, and data interpretation by protein bioinformatics.-->
                            <!--</p>-->
                            <!--<p style="font-weight: 500">-->
                            <!--Ref: Kenyo, G, et al., Mol. Cell Proteomics 2002,1:763-80. PMID: 12438560-->
                            <!--</p>-->


                            <a name="Proteotypic-peptides"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Proteotypic peptides</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to peptides that uniquely represent targeted proteins or protein isoforms.
                                Identification of proteotypic peptides of target proteins is a pre-requisite step of
                                <a href="#Targeted-proteomics">targeted proteomics</a> workflow. Lists of
                                proteotypic peptides for any particular proteins can be obtained from own proteomic
                                experiments, predicted by using computational tools or retrieved from public
                                databases, i.e., SRMAtlas.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Keerthikumar S et al. Methods Mol Biol. 2017;1549:101-107. PMID: 27975286
                            </p>


                            <a name="Q1"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Q1</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the mass filter for particular precursor ions
                            </p>


                            <a name="Q3"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Q3</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the mass filter for particular fragment ions
                            </p>


                            <a name="Quantitative-proteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Quantitative proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                There are two main approaches of quantitative proteomics, including the stable
                                isotope-based or label-free methods. Isotope-based methods incorporate heavy
                                versions of specific molecules into the peptides, either by chemical derivatization
                                (i.e., <a href="#iTRAQ">iTRAQ</a>, <a href="#TMT">TMT</a>) or by metabolic labeling
                                (e.g., <a href="#SILAC">SILAC</a>). Label-free methods (such as the spectral
                                counting and label-free quantitation) have less accurate peptide quantification than
                                the isotope-based strategies. However, label-free methods are robust and
                                cost-effective, giving these strategies are increasingly attractive for large-scale
                                proteomic studies.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Cox J and Mann M, Annu Rev Biochem. 2011;80:273-99. PMID: 21548781
                            </p>


                            <a name="Retention-time-(RT)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Retention time (RT)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a particular time of a specific peptide eluted from a <a href="#LC">LC</a>
                                run.
                            </p>


                            <a name="SDS"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">SDS</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to Sodium dodecyl sulphate, an ionic detergent constitutes in lysis buffers
                                used for protein solubilization.
                            </p>


                            <a name="Sequence-tag-identification"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Sequence tag identification</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                This approach leads to protein identification by combining mass measurement and
                                short amino acid sequence information obtained by tandem <a
                                    href="#Mass-spectrometry">mass spectrometry</a>. This information is then used
                                to automatically find the best match in a sequence database. The procedure is
                                repeated with several peptides from the digest, resulting in multiple
                                identifications of the same protein or identification of several proteins from the
                                peptide mixture.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Controlled vocabularies, HUPO Proteomics Standard Initiative (last maintenance
                                and update Oct 2015) (www.psidev.info/groups/controlled-vocabularies)
                            </p>


                            <a name="Shotgun-proteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Shotgun proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Shotgun proteomics is the high-throughput protein identification in peptide mixture
                                obtained by the proteolytic digestion of complex samples. Shotgun proteomics is the
                                most widely used and the most generic approach of <a href="#Mass-spectrometry">mass
                                spectrometry</a> based proteomics.
                                For detail information, please see <a href="#Bottom-up-proteomics">"Bottom-up
                                proteomics"</a>, <a href="#Discovery-proteomics">“Discovery proteomics”</a> and <a
                                    href="#Data-dependent-acquisition-(DDA)">“Data dependent acquisition”</a>.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Aebersold R and Mann M, Nature 422:198–207. PMID: 12634793
                            </p>


                            <a name="Skyline"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Skyline</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Skyline is a freely-available, open-source Windows client application for building
                                <a href="#Single-Reaction-Monitoring (SRM)">SRM</a>/<a
                                    href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a>, <a
                                    href="#Parallel-Reaction-Monitoring-(PRM)">PRM</a>, <a
                                    href="#Data-independent-acquisition-(DIA)">DIA,</a><a href="#SWATH-MS">SWATH</a>
                                and targeted <a href="#Data-dependent-acquisition-(DDA)">DDA</a> quantitative
                                methods and analyzing the resulting mass spectrometer data.

                                For more information, please see
                                https://skyline.ms/project/home/software/Skyline/begin.view
                            </p>


                            <a name="SIL-internal-standards"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">SIL internal standards</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to stable isotope labelled analog of the analyte. In the SIL internal
                                standards, several atoms in the analyte are replaced by their stable isotopes, such
                                as deuterium (D), 13C, 15N, or 18O. SIL peptides are typically used as the internal
                                standards to generate the calibration curve for precise and accurate quantitation of
                                targeted proteins in <a href="#Mass-spectrometry">mass spectrometry</a> proteomic
                                assay.
                            </p>


                            <a name="SILAC"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">SILAC</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to stable isotope labeling using amino acids in cell culture. SILAC is a
                                metabolic labeling method that incorporates “heavy” 13C- or 15N-labeled amino acids
                                into cellular proteins in vivo for quantitative proteomics.

                                For more information, please see
                                “https://www.thermofisher.com/us/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/silac-metabolic-labeling-systems.html”
                            </p>


                            <a name="Single-Reaction-Monitoring (SRM)"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Single Reaction Monitoring (SRM)</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                SRM is a targeted proteomic approach. SRM primarily performs on triple quadrupole
                                (QqQ) <a href="#Mass-spectrometry">mass spectrometry</a> and serves as the MS
                                “gold-standard” for <a href="#Targeted-proteomics">targeted proteomics</a>. <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a> is a very sensitive and highly
                                selective method to detect and quantify a target peptide, and by inference, a target
                                protein in complex samples. When applying multiple targets, a term of <a
                                    href="#Multiple-Reaction-Monitoring-(MRM)">multiple reaction monitoring
                                (MRM)</a> is used to represent multiplexing capability of <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a>.
                            </p>
                            <h2 style="text-decoration: underline;">Technique:</h2>
                            <p class="fh5co-sub-lead">
                                SRM utilizes a unique capability of the Triple Quadrapole (QQQ) mass spectrometer to
                                act as mass filters (<a href="#Q1">Q1</a>) which then isolate the target analyte
                                (precursor ion) from the contaminants. The precursor ion is sent to fragmentation in
                                Q2 by collision-induced dissociation. A specific fragment ion from the precursor ion
                                is selected in <a href="#Q3">Q3</a> and guided to the detector. The number of
                                specific fragment ions that reach the detector is counted over time, resulting in a
                                chromatographic trace with <a href="#Retention-time-(RT)">retention time</a> and
                                signal intensity as coordinates. Monitoring several precursor-fragment ion pairs (or
                                <a href="#Transitions">transitions</a>) yields the chromatographic peaks at a
                                specific <a href="#Retention-time-(RT)">retention time</a> that supports relative or
                                absolute (if an appropriate SIS-peptide is used) quantification of a targeted
                                peptide. A set of suitable <a href="#Transitions">transitions</a> can be applied to
                                generate a specific assay to detect and quantify the target peptide, and by
                                inference, the target protein in complex samples.
                            </p>
                            <h2 style="text-decoration: underline;">Applications:</h2>
                            <p class="fh5co-sub-lead">
                                1. Study of protein modifications. For example, SRM/<a
                                    href="#Multiple-Reaction-Monitoring-(MRM).">MRM</a> can be used to measure
                                abundance changes of phosphorylated proteins to a series of pertubations in multiple
                                cancer cell lines which can give insights into functional changes in the protein
                                pathways and networks responsible to the pertubagens (<a href=""
                                                                                         ng-click="scrollTo('P100Info')">P100</a>
                                set in the iLINC project).
                            </p>
                            <p class="fh5co-sub-lead">
                                2. Biology and diseases. SRM/<a href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a>
                                can be used to accurately quantify a series of 50-100 proteins involved in cellular
                                biological pathways at different functional/pathological states.
                            </p>
                            <p class="fh5co-sub-lead">
                                3. Clinical application. SRM/<a href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a>
                                is well-suited for biomarker validation of multiple target proteins across large
                                sample sets of clinical relevant specimens, i.e., plasma, serum and urine. Future
                                developments of SRM/<a href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a> as a
                                platform of multiplexing clinical assays is promising.
                            </p>
                            <h2 style="text-decoration: underline;">Limitations:</h2>
                            <p class="fh5co-sub-lead">
                                : Number of measurable proteins is limited to 50-100 proteins per one <a
                                    href="#LC">LC</a>-MS
                                run. Accurate quantification of low-abundant proteins in complex samples is also
                                limited by the sensitivity of SRM measurement, unless sample prefractionation or
                                enrichment steps are applied.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Picotti P, et al. Nat Methods. 2012;9(6):555-66. PMID: 22669653
                            </p>


                            <a name="SWATH-MS"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">SWATH-MS</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to Sequential Window Acquisition of all THeoretical fragment ion-<a
                                    href="#Mass-spectrometry">mass spectrometry</a>. SWATH-MS is a targeted
                                label-free proteomic approach that couples <a
                                    href="#Data-independent-acquisition-(DIA)">DIA</a> with direct searching of
                                individual samples against the established spectral library. A major advantage is
                                that SWATH-MS can maximize peptide detection at a comparable coverage of <a
                                    href="#Shotgun-proteomics">shotgun proteomics</a> while still quantitates
                                peptide amount with consistency and reproducibility across various samples as <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a>/<a
                                    href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a> and <a
                                    href="#Parallel-Reaction-Monitoring-(PRM)">PRM</a>, thus increasing experimental
                                efficiency and reducing quantitative variability in the proteomic study.
                            </p>
                            <img src="/lincsproteomics/images/7.png"
                                 class="centered">
                            <!--<img border="0" src="/lincsproteomics/images/7.png" width="4%" height="4%"-->
                            <!--style="align-content: center; ">-->
                            <p class="fh5co-sub-lead">
                                (modified from www.sciex.com)
                            </p>
                            <h2 style="text-decoration: underline;">Technique:</h2>
                            <p class="fh5co-sub-lead">
                                SWATH-MS is a 2-step process; first, the spectral library generation, and second, <a
                                    href="#SWATH-MS">SWATH</a> data extraction. The spectral libraries can be simply
                                built using the information of <a href="#Data-dependent-acquisition-(DDA).">DDA</a>
                                data of samples of interest in which the same set of samples is planned for further
                                analysis in <a href="#Data-independent-acquisition-(DIA)">DIA</a> mode. This will
                                give a very basic spectral library for <a href="#SWATH-MS">SWATH</a> data extraction.
                                Alternatively, information from multiple <a
                                    href="#Data-dependent-acquisition-(DDA).">DDA</a> runs of technical replicates
                                and/or samples obtained from different genotypes, phenotypes, perturbations and
                                fractionation strategies will help increase the depth of proteome coverage and more
                                inclusive spectral libraries.
                                For <a href="#SWATH-MS">SWATH</a> data extraction, the <a
                                    href="#Data-independent-acquisition-(DIA)">DIA</a> data are searched against the
                                generated spectral library that allows a set number of fragment ion chromatograms to
                                be extracted for a peptide within the window of its predicted <a
                                    href="#Retention-time-(RT)">retention time</a>. The peak groups are scored to
                                discriminate a “true” peptide target from nonspecific noise and distribution of
                                these target scores is modeled against the distribution of scores attributed to
                                decoy peak groups to determine a score cut off, resulting in an acceptable <a
                                    href="#False-discovery-rate-(FDR)">false discovery rate</a> using the
                                target-decoy approach. Peptide abundance is then inferred from the aggregate of the
                                area under the curve for each fragment ion extracted ion chromatograms (XICs), which
                                can be used to estimate the overall protein intensity.
                            </p>
                            <h2 style="text-decoration: underline;">Applications:</h2>
                            <p class="fh5co-sub-lead">
                                SWATH-MS is a suitable platform for any proteomic project that need to accurately
                                quantify 1000s peptides across a large number of samples, therefore serving for both
                                discovery and targeted proteomic experiments.
                            </p>
                            <h2 style="text-decoration: underline;">Limitations:</h2>
                            <p class="fh5co-sub-lead">
                                SWATH-MS has less selective and sensitive than <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a> and <a
                                    href="#Parallel-Reaction-Monitoring-(PRM)">PRM</a>. Data analysis always needs
                                sophisticate analytical pipeline and particular software as compared to <a
                                    href="#Data-dependent-acquisition-(DDA)">DDA</a>, <a
                                    href="#Single-Reaction-Monitoring (SRM)">SRM</a>/<a
                                    href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a> or <a
                                    href="#Parallel-Reaction-Monitoring-(PRM)">PRM</a>.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Holewinski RJ et al., Methods Mol Biol. 2016;1410:265-79. PMID: 26867750
                            </p>


                            <a name="Targeted-proteomics" />
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px; ">Targeted proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to a proteomic approach which specifies targets in complex protein mixture
                                and determines their presence and quantity across a sample cohort. There are three
                                main methods including <a href="#Single-Reaction-Monitoring (SRM)">SRM</a>/<a
                                    href="#Multiple-Reaction-Monitoring-(MRM)">MRM</a>, <a
                                    href="#Parallel-Reaction-Monitoring-(PRM)">PRM</a>, and <a
                                    href="#SWATH-MS">SWATH</a>.
                            </p>
                            <img src="/lincsproteomics/images/8.png"
                                 class="centered-wider">
                            <!--<img border="0" src="/lincsproteomics/images/8.png" width="20%" height="20%"-->
                            <!--style="align-content: center; ">-->
                            <p style="font-weight: 500">
                                Ref: Vidova V et al. Anal Chim Acta. 2017 Apr 29;964:7-23. PMID: 28351641
                                Shi T et al, Proteomics. 2016 Aug; 16(15-16): 2160–2182. PMID: 27302376
                            </p>


                            <a name="TCEP"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">TCEP</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to tris(2-carboxyethyl)phosphine, an irreversible reducing agent that can use
                                to break disulfide bonds of proteins.
                            </p>


                            <a name="TMT"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">TMT</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to Tandem Mass Tag reagent. The TMT reagent can be used to enable protein
                                identification and quantitation of up to 10 different samples at a single MS/MS
                                experiment.

                                For more information, please see
                                “www.thermofisher.com/us/en/home/life-science/protein-biology/protein-mass-spectrometry-analysis/protein-quantitation-mass-spectrometry/tandem-mass-tag-systems.html”
                            </p>


                            <a name="Top-down-proteomics"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Top-down proteomics</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the study of the <a href="#Proteoforms)">proteoforms</a> - all molecular
                                forms in which the protein products of a single gene can be found including all
                                forms of genetic variation, alternative splicing of RNA transcripts and <a
                                    href="#Post-translational-modification-(PTM)">post-translational
                                modifications</a>. As <a href="#Proteoforms">proteoforms</a> are tightly linked the
                                functions of cells and tissues underlie the complex phenotypes, top-down proteomics
                                will provide critical insights into fundamental process of biological systems.
                            </p>
                            <img src="/lincsproteomics/images/9.png"
                                 class="centered-wider">
                            <!--<img border="0" src="/lincsproteomics/images/9.png" width="20%" height="20%"-->
                            <!--style="align-content: center; ">-->
                            <p style="font-weight: 500">
                                Ref: Toby TK, et al., Annu Rev Anal Chem. 2016;9(1):499–519. PMID: 27306313
                                Smith LM et al. Science. 2018;359(6380)1106-1107. PMID: 29590032
                            </p>


                            <a name="Transitions"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Transitions</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                Refers to the specific pair of m/z values associated to the precursor and the
                                selected fragment ions. Transition can be written as parent m/z > fragment m/z (e.g.
                                440.7 > 201.1).
                            </p>
                            <p style="font-weight: 500">
                                Source: http://www.mrmatlas.org/glossary2.php
                            </p>


                            <a name="Trypsin"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Trypsin</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                The most common proteolytic enzyme uses in bottom up proteomic experiments. This
                                enzyme cleaves at C-terminus of lysine or arginine, producing peptides from a
                                protein. Note that trypsin cannot cleave lysines follow by prolines, resulting in
                                miss cleavage peptides.
                            </p>


                            <a name="Ubiquitin-modification"/>
                            <div style="background-color:#eef2d7;padding-top: 60px">
                                <h3 class="onerow" style="padding-bottom: 0px">Ubiquitin modification</h3>
                                <a class="onerow" style="float:right; padding-bottom: 0px" href=""
                                   ng-click="scrollTo('top')">Back to top</a>
                            </div>

                            <p class="fh5co-sub-lead">
                                A <a href="#Post-translational-modification-(PTM)">post-translational
                                modification</a> involving a sequential reaction steps of E1/E2/E3 enzymes leading
                                to ubiquitin activation and transferring to the substrate proteins. Ubiquitin
                                modification (or ubiquitination) starts by the attachment of the 76-amino acid
                                protein ubiquitin to a substrate lysine residue via an isopeptide bond, resulting in
                                ubiquitinated lysine. Ubiquitin modification can occur in mono-ubiquitination and
                                poly-ubiquitination, while poly-ubiquitination can be homotypic and heterotypic
                                linkage types.
                                After tryptic digestion, the ubiquitin is cleaved as same as the substrate protein
                                by the enzyme <a href="#Trypsin">trypsin</a>, resulting in a diglycine remnant of
                                ubiquitin remained on the lysine residue (GG--K) of the substrate peptide. The
                                GG--K is a basis of ubiquitin modified peptide enrichment and bottom-up proteomic
                                analysis.
                            </p>
                            <p style="font-weight: 500">
                                Ref: Swatek KN et. al. Cell Res. 2016;26(4):399-422. PMID: 27012465
                            </p>


                        </div>
                        <!--</div>-->
                        <!--<div class=tab-content>-->
                    </div>
                    <div class="tab-pane fade " id="about">
                        <div class="row ">
                            <div class="col-sm-5">
                                <figure>
                                    <img src="/lincsproteomics/images/all-to-gether.png"
                                         class="img-responsive" width="450px" height="450px">
                                </figure>
                            </div>
                            <div class="col-sm-7">
                                <!--<div class="fh5co-label animate-box">Who We Are:</div>-->
                                <h2 class="fh5co-lead ">LINCS Proteomics Landing Page</h2>
                                <p class="fh5co-sub-lead ">
                                    <!--<div class="container" >-->
                                    <!--<h2></h2>-->
                                    <!--<p >-->

                                    LINCS proteomics portal has been developed by the LINCS Data Coordination
                                    and Integration Center
                                    (DCIC) in collaboration with other LINCS centers, and with the goal of
                                    facilitating the
                                    navigation through and interaction with the LINCS proteomics data.
                                    Your feedback is highly appreciated and please

                                    <a
                                            href="mailto:behrouz.shamsaei@uc.edu?subject=Support">contact our
                                        team using this link.</a>

                                </p>

                                <!--<hr/>-->
                            </div>
                            <div class="col-sm-12">
                                <h2 class="fh5co-lead ">The LINCS-Proteomics portal aims to:</h2>
                                <p class="fh5co-sub-lead ">
                                    <i class="fa fa-gear"></i>&nbsp; Facilitate
                                    navigation, search, and interpretation of LINCS proteomics data using
                                    alternative
                                    “data-centric” and “assay-centric” lens or views<br/>

                                    <i class="fa fa-gear"></i> &nbsp; Provide a guide
                                    to LINCS Data Portal other resources developed by LINCS-DCIC to facilitate
                                    finding and
                                    downloading proteomic data<br/>

                                    <i class="fa fa-gear"></i>&nbsp; Provide intuitive
                                    representations of relationship between protein entities that are measured
                                    as the snapshot
                                    of the biological state provided by each method/assay<br/>

                                    <i class="fa fa-gear"></i>&nbsp; Provide links to
                                    other resources and tools for proteomics, in particular those developed as
                                    part of the LINCS
                                    consortium<br/>

                            </div>
                        </div>
                    </div>
                    <!--</div>-->
                    <!--<div class=tab-content>-->

                    <div class="tab-pane fade" id="centers">
                        <div class="row animate-box">
                            <div class="col-md-6">
                                <figure>
                                    <img src="/lincsproteomics/images/LINCS.png"
                                         alt="Free HTML5 Bootstrap Template by FREEHTML5.co"
                                         class="img-responsive">
                                </figure>
                            </div>
                            <div class="col-md-6">
                                <!--<div class="fh5co-label animate-box">Who We Are:</div>-->
                                <h2 class="fh5co-lead ">The Library of Integrated Network-Based Cellular
                                    Signatures
                                    (LINCS) </h2>

                                <p class="fh5co-sub-lead ">LINCS program aims to create a network-based
                                    understanding
                                    of biology by cataloging changes in gene expression and other cellular
                                    processes that occur
                                    when cells are exposed to a variety of perturbing agents.</p>
                                <ul class="fh5co-social">
                                    <li><a href="https://twitter.com/LINCSProgram" target="_blank"><i
                                            class="icon-twitter"></i></a></li>

                                    <li><a href="http://www.lincsproject.org/" target="_blank"><i
                                            class="icon-globe"></i></a></li>

                                </ul>
                                <div class="onerow">
                                    <a href="" ng-click="showCenterInfo = !showCenterInfo" class=" a green ">More
                                        Information About
                                        LINCS
                                        Proteomics Centers</a>
                                    <div ng-show="!showCenterInfo" ng-click="showCenterInfo = !showCenterInfo">
                                            <span class="a green arrow" style="size: 10px; "><i
                                                    class="icon-chevron-down"></i></span>
                                    </div>
                                    <div ng-show="showCenterInfo" ng-click="showCenterInfo = !showCenterInfo">
                                                <span class=" arrow green" style="size: 10px; "><i
                                                        class="icon-chevron-up"></i></span>
                                    </div>
                                </div>
                            </div>
                        </div>


                        <div ng-show="showCenterInfo" class=" animate-box">
                            <hr/>
                            <div class="col-md-6 col-md-offset-3 text-center">

                                <h2 class="fh5co-lead " style="font-size: 30px">LINCS Proteomics Data
                                    Centers </h2>
                                <div class="fh5co-label ">This list includes data generating and data
                                    processing centers
                                </div>
                            </div>
                            <div class="row">

                                <div class="col-md-3 ">
                                    <div class="fh5co-person">
                                        <figure class="">
                                            <img src="/lincsproteomics/images/hms-lincs2.png"
                                                 alt="Free HTML5 Bootstrap Template by FREEHTML5.co"
                                                 class="img-responsive" width="200px" height="100px">
                                        </figure>
                                        <h3 class="fh5co-name ">HMS LINCS</h3>
                                        <h4 class="fh5co-designation ">Harvard Medical School</h4>
                                        <p class="fh5co-bio ">The HMS LINCS Center collects multiplex
                                            perturbagen-response data across a
                                            number of complementary
                                            experimental platforms including live- and fixed-cell imaging,
                                            sandwich
                                            immunoassays, protein mass
                                            spectrometry, and mRNA profiling as a means to measure the
                                            levels.</p>
                                        <ul class="fh5co-social">
                                            <!--<li class="animate-box"><a href="#"><i class="icon-twitter"></i></a></li>-->
                                            <!--<li class="animate-box"><a href="#"><i class="icon-dribbble"></i></a></li>-->
                                            <li class=""><a href="http://lincs.hms.harvard.edu/"
                                                            target="_blank"><i
                                                    class="icon-globe"></i></a></li>
                                        </ul>
                                    </div>
                                </div>
                                <div class="col-md-3 col-md-offset-1">
                                    <div class="fh5co-person">
                                        <figure>
                                            <img src="/lincsproteomics/images/dtoxs2.png"
                                                 alt="Free HTML5 Bootstrap Template by FREEHTML5.co"
                                                 class="img-responsive" width="200px" height="100px">
                                        </figure>
                                        <h3 class="fh5co-name ">LINCS DToxS</h3>
                                        <h4 class="fh5co-designation animate-box">Icahn School of Medicine</h4>
                                        <p class="fh5co-bio "> The Drug Toxicity Signature Generation Center is
                                            a
                                            Systems Pharmacology
                                            research
                                            center at the Icahn School of Medicine. The proteomics experiments
                                            are
                                            conducted at
                                            the Center for Advanced Proteomics, Rutgers-New Jersey Medical
                                            School.</p>
                                        <ul class="fh5co-social">
                                            <!--<li class="animate-box"><a href="#"><i class="icon-twitter"></i></a></li>-->
                                            <!--<li class="animate-box"><a href="#"><i class="icon-github"></i></a></li>-->
                                            <li><a href="https://martip03.u.hpc.mssm.edu/"
                                                   target="_blank"><i class="icon-globe"></i></a></li>
                                        </ul>
                                    </div>
                                </div>
                                <div class="col-md-3 col-md-offset-1">
                                    <div class="fh5co-person">
                                        <figure>
                                            <img src="/lincsproteomics/images/neurolincs2.jpeg"
                                                 alt="Free HTML5 Bootstrap Template by FREEHTML5.co"
                                                 class="img-responsive"
                                                 width="200px" height="100px">
                                        </figure>
                                        <h3 class="fh5co-name ">NeuroLINCS</h3>
                                        <h4 class="fh5co-designation ">University of California- Irvine, Cedars
                                            Sinai, the Gladstone Institute, MIT, and Johns Hopkins
                                            University</h4>
                                        <p class="fh5co-bio ">The NeuroLINCS Center studies ALS and SMA by
                                            characterizing the molecular
                                            networks within
                                            patient-derived iPSCs, and their differentiated progeny, across many
                                            regulatory layers.</p>
                                        <ul class="fh5co-social">
                                            <li><a href="https://twitter.com/neurolincs" target="_blank"><i
                                                    class="icon-twitter"></i></a></li>
                                            <li><a href="https://github.com/neurolincs" target="_blank"><i
                                                    class="icon-github"></i></a></li>
                                            <li><a href="http://neurolincs.org/technologies/"
                                                   target="_blank"><i class="icon-globe"></i></a></li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                            <hr/>
                            <div class="row">
                                <div class="col-md-6 ">
                                    <div class="fh5co-person">
                                        <figure>
                                            <img src="/lincsproteomics/images/panorama-logo.png"
                                                 alt="Free HTML5 Bootstrap Template by FREEHTML5.co"
                                                 class="img-responsive"
                                                 width="300px" height="150px">
                                        </figure>
                                        <h3 class="fh5co-name ">LINCS PCCSE</h3>
                                        <h4 class="fh5co-designation ">Broad Institute</h4>
                                        <p class="fh5co-bio ">The LINCS Proteomic Characterization Center for
                                            Signaling and Epigenetics at
                                            the
                                            Broad Institute/MIT/University of Washington utilizes the P100 and
                                            GCP
                                            assays to
                                            profile the response of human cell lines to perturbations.</p>
                                        <ul class="fh5co-social">
                                            <!--<li class="animate-box"><a href="#"><i class="icon-twitter"></i></a></li>-->
                                            <!--<li class="animate-box"><a href="#"><i class="icon-dribbble"></i></a></li>-->
                                            <li><a href="https://panoramaweb.org/project/LINCS/begin.view"
                                                   target="_blank"><i class="icon-globe"></i></a></li>
                                        </ul>
                                    </div>
                                </div>
                                <div class="col-md-6 ">
                                    <div class="fh5co-person">
                                        <figure>
                                            <img src="/lincsproteomics/images/DCIC_BD2K.png"
                                                 alt="Free HTML5 Bootstrap Template by FREEHTML5.co"
                                                 class="img-responsive" width="200px" height="200px">
                                        </figure>
                                        <h3 class="fh5co-name ">BD2K-LINCS DCIC</h3>
                                        <h4 class="fh5co-designation ">Icahn School of Medicine at Mount Sinai,
                                            University of Cincinnati and University of Miami</h4>
                                        <p class="fh5co-bio ">The BD2K-LINCS Data Coordination and Integration
                                            Centers at Icahn School of
                                            Medicine, university of Miami
                                            and university of Cincinnati are responsible for harmonizing
                                            LINCS data with other available resources.</p>
                                        <ul class="fh5co-social">
                                            <li><a href="https://twitter.com/BD2KLINCSDCIC" target="_blank"><i
                                                    class="icon-twitter"></i></a></li>

                                            <li><a href="http://lincs-dcic.org/" target="_blank"><i
                                                    class="icon-globe"></i></a></li>
                                        </ul>
                                    </div>
                                </div>
                            </div>
                        </div>


                        <div id="fh5co-testimonials2" style="padding-top: 15px;">
                            <!--<a name="centersInfo" />-->


                            <div style="padding-top: 15px;">


                                <!--<h2 class="fh5co-lead animate-box text-center">LINCS Proteomics Centers</h2>-->

                                <div class="row ">
                                    <div class="col-md-6 col-md-offset-3 text-center">

                                        <h2 class="fh5co-lead " style="font-size: 30px">LINCS Proteomics Data
                                            Centers </h2>
                                        <div class="fh5co-label ">This list includes data generating and data
                                            processing centers
                                        </div>
                                    </div>
                                </div>
                                <div class="row">
                                    <div class="col-md-8 col-md-offset-2 text-center animate-box">
                                        <div class="flexslider">
                                            <ul class="slides">
                                                <li>
                                                    <blockquote>
                                                        <h3>LINCS PCCSE </h3>
                                                        <p>The LINCS Proteomic Characterization Center for
                                                            Signaling
                                                            and
                                                            Epigenetics at
                                                            the
                                                            Broad Institute/MIT/University of Washington
                                                            utilizes
                                                            the P100 and
                                                            GCP
                                                            assays to
                                                            profile the response of human cell lines to
                                                            perturbations.</p>
                                                        <p><cite>&nbsp;</cite></p>
                                                    </blockquote>
                                                </li>
                                                <li>
                                                    <blockquote>
                                                        <h3>LINCS DToxS </h3>
                                                        <p>The Drug Toxicity Signature Generation Center is a
                                                            Systems
                                                            Pharmacology
                                                            research
                                                            center at the Icahn School of Medicine. The
                                                            proteomics
                                                            experiments
                                                            are
                                                            conducted at
                                                            the Center for Advanced Proteomics, Rutgers-New
                                                            Jersey
                                                            Medical
                                                            School.</p>
                                                        <p><cite>&nbsp;</cite></p>
                                                    </blockquote>
                                                </li>
                                                <li>
                                                    <blockquote>
                                                        <h3>HMS LINCS </h3>
                                                        <p>The HMS LINCS Center collects multiplex
                                                            perturbagen-response data
                                                            across a
                                                            number of complementary
                                                            experimental platforms including live- and
                                                            fixed-cell
                                                            imaging,
                                                            sandwich
                                                            immunoassays, protein mass
                                                            spectrometry, and mRNA profiling as a means to
                                                            measure
                                                            the levels,
                                                            localization, and states of modification.</p>
                                                        <p><cite>&nbsp;</cite></p>
                                                    </blockquote>
                                                </li>
                                                <li>
                                                    <blockquote>
                                                        <h3>NeuroLINCS </h3>
                                                        <p>The NeuroLINCS Center studies ALS and SMA by
                                                            characterizing the
                                                            molecular
                                                            networks within
                                                            patient-derived iPSCs, and their differentiated
                                                            progeny,
                                                            across many
                                                            regulatory layers.</p>
                                                        <p><cite>&nbsp;</cite></p>
                                                    </blockquote>
                                                </li>
                                                <li>
                                                    <blockquote>
                                                        <h3>BD2K-LINCS DCIC </h3>
                                                        <p>The BD2K-LINCS Data Coordination and Integration
                                                            Centers
                                                            at Icahn
                                                            School of
                                                            Medicine, university of Miami
                                                            and university of Cincinnati are responsible for
                                                            harmonizing
                                                            LINCS data with other available resources.</p>
                                                        <p><cite>&nbsp;</cite></p>
                                                    </blockquote>
                                                </li>
                                            </ul>
                                        </div>
                                        <div class="flexslider-controls" style="padding-top: 10px">
                                            <ol class="flex-control-nav">
                                                <li class="animate-box"><img
                                                        src="/lincsproteomics/images/labkey-logo2.png"
                                                        alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                                </li>
                                                <li class="animate-box"><img
                                                        src="/lincsproteomics/images/dtoxs2.png"
                                                        alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                                </li>
                                                <li class="animate-box"><img
                                                        src="/lincsproteomics/images/hms-lincs2.png"
                                                        alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                                </li>
                                                <li class="animate-box"><img
                                                        src="/lincsproteomics/images/neurolincs2.jpeg"
                                                        alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                                </li>
                                                <li class="animate-box"><img
                                                        src="/lincsproteomics/images/dcic3.png"
                                                        alt="Free HTML5 Bootstrap Template by FREEHTML5.co">
                                                </li>
                                            </ol>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div>

                        <!--<div id="fh5co-pricing">-->
                        <!--<div style="padding-top: 30px">-->

                            <!--<iframe src="https://www.google.com/maps/d/embed?mid=1MhDhNDsvCJvAmSNypmyCaj-g2EQ&hl=en"-->
                                    <!--width="100%"-->
                                    <!--height="700"></iframe>-->
                          <!---->
                        <!--</div>-->


                        <!--<a name="centersInfo"></a>-->


                        <!--<h3 class="animate-box"></h3>-->
                        <!--<hr/>-->


                        <!--</div>-->


                    </div>
                    <!--</div>-->
                    <!--<div class=tab-content>-->
                    <div class="tab-pane fade" id="publication">
                        <h3 class="fh5co-lead " style="background-color:#929971">Publications:</h3>
                        <div class="fh5co-label ">For complete list of LINCS consortium publications please
                            visit <a
                                    href="http://www.lincsproject.org/LINCS/publications"
                                    target="_blank">here</a>.
                        </div>

                        <h2 class="fh5co-lead " style="background-color:#d1d8ab">Featured LINCS
                            Publications:</h2>

                        <div class="fh5co-sub-lead ">
                            <a ng-href="http://www.lincsproject.org/" target="_blank">
                                <img border="0" src="/lincsproteomics/images/DCIC_BD2K.png" width="7%"
                                     height="7%">
                            </a>&nbsp;

                            Keenan AB, et al [...] Pillai A, 2017. <a
                                href="https://www.ncbi.nlm.nih.gov/pubmed/29199020"
                                target="_blank">The Library of Integrated Network-Based Cellular Signatures NIH
                            program: System-level cataloging of human cells response to perturbations.</a> Cell
                            Systems. pii: S2405-4712(17):30490-30498.
                            <br/>
                            <hr/>
                        </div>


                        <div class="fh5co-sub-lead ">
                            <a ng-href="http://lincsportal.ccs.miami.edu/dcic-portal/" target="_blank">
                                <img border="0" src="/lincsproteomics/images/ldp_logo.png" width="5%"
                                     height="5%">
                            </a>&nbsp;


                            Koleti A, Terryn R, Stathias V, Chung C, Cooper DJ, Turner JP, Vidovic D, Forlin M,
                            Kelley TT, D'Urso A, Allen BK, Torre D, Jagodnik KM, Wang L, Jenkins SL, Mader C,
                            Niu W, Fazel M, Mahi M, Pilarczyk M, Clark N, Shamsaei B, Meller J, Vasiliauskas J,
                            et al. 2017. <a href="https://academic.oup.com/nar/article/46/D1/D558/4621325"
                                            target="_blank">Data Portal for the Library of Integrated
                            Network-based Cellular Signatures (LINCS) program: integrated access to diverse
                            large-scale cellular perturbation response data.</a> Nucleic Acids Research.
                            gkx1063.<br/>
                            <hr/>
                        </div>


                        <h2 class="fh5co-lead " style="background-color:#d1d8ab">Featured LINCS Proteomics
                            Publications:</h2>

                        <div class="fh5co-sub-lead ">
                            <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                            Koplev S, Lin K, Dohlman AB, Ma'ayan A. 2018. <a
                                href="https://www.ncbi.nlm.nih.gov/pubmed/29293502"
                                target="_blank">Integration of pan-cancer
                            transcriptomics with RPPA proteomics
                            reveals mechanisms of epithelial-mesenchymal transition.</a> PLoS Computational
                            Biology.
                            14(1):e1005911.<br/>
                            <hr/>
                        </div>


                        <div class="fh5co-sub-lead ">
                            <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                            Erickson BK, Rose CM, Braun CR, Erickson AR, Knott J, McAlister GC, Wühr M, Paulo
                            JA, Everley RA,
                            Gygi
                            SP. 2017. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28065596" target="_blank">A
                            strategy to
                            combine
                            sample multiplexing with targeted proteomics assays for high-throughput protein
                            signature
                            characterization.</a> Molecular Cell. 65(2):361-370.
                            <br/>
                            <hr/>
                        </div>


                        <div class="fh5co-sub-lead ">
                            <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                            Litichevskiy L, Peckner R, Abelin JG, Asiedu JK, Creech AL, Davis JF, Davison D,
                            Dunning CM,
                            Egertson
                            JD, Egri S, Gould J, et al [...] Jaffe J. 2017. <a
                                href="https://www.biorxiv.org/content/early/2017/09/07/185918" target="_blank">A
                            library of
                            phosphoproteomic and chromatin signatures for characterizing cellular responses to
                            drug
                            perturbations.</a> bioRxiv.
                            <br/>
                            <hr/>
                        </div>


                        <div class="fh5co-sub-lead ">
                            <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                            Everley RA, Huttlin EL, Erickson AR, Beausoleil SA, Gygi SP. 2016. <a
                                href="https://www.ncbi.nlm.nih.gov/pubmed/27978624" target="_blank">Neutral loss
                            is a very
                            common
                            occurrence in phosphotyrosine-containing peptides labeled with isobaric tags.</a>
                            Journal of
                            Proteome
                            Research. 16(2):1069-1076.
                            <br/>
                            <hr/>
                        </div>


                        <div class="fh5co-sub-lead ">
                            <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                            Parker SJ, Venkatraman V, Van Eyk JE. 2016. <a
                                href="https://www.ncbi.nlm.nih.gov/pubmed/27432805"
                                target="_blank">Effect of peptide assay library size
                            and
                            composition in targeted data-independent acquisition-MS analyses.</a> Proteomics.
                            16(15-16):2221-37.
                            <br/>
                            <hr/>
                        </div>
                        <div ng-show="!ctrl.showAllPapers">
                            <button class="btn btn-info" ng-click="ctrl.showAllPapers = true">Show All Papers
                            </button>
                        </div>
                        <div ng-show="ctrl.showAllPapers">
                            <button class="btn btn-info" ng-click="ctrl.showAllPapers = false">Show Less
                            </button>
                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Shi T, Niepel M, McDermott JE, Gao Y, Nicora CD, Chrisler WB, Markillie LM,
                                Petyuk VA, Smith RD,
                                Rodland
                                KD, Sorger PK, Qian WJ, Wiley HS. 2016. <a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/27405981"
                                    target="_blank">Conservation of protein abundance
                                patterns
                                reveals the regulatory architecture of the EGFR-MAPK pathway.</a> Science
                                Signaling. 9(436):rs6.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG,
                                Ma'ayan A. 2016. <a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/27374120" target="_blank">The
                                Harmonizome: a
                                collection of
                                processed datasets gathered to serve and mine knowledge about genes and
                                proteins.</a> Database
                                (Oxford).
                                baw100.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Zhang YJ, Gendron TF, Grima JC, Sasaguri H, Jansen-West K, Xu YF, Katzman RB,
                                Gass J, Murray ME,
                                Shinohara M, Lin WL, Garrett A, Stankowski JN, Daughrity L, Tong J, Perkerson
                                EA, Yue M, Chew J,
                                Castanedes-Casey M, Kurti A, Wang ZS, Liesinger AM, Baker JD, Jiang J,
                                Lagier-Tourenne C, Edbauer D,
                                Cleveland DW, Rademakers R, Boylan KB, Bu G, Link CD, Dickey CA, Rothstein JD,
                                Dickson DW, Fryer JD,
                                Petrucelli L. 2016.<a href="https://www.ncbi.nlm.nih.gov/pubmed/26998601"
                                                      target="_blank"> C9ORF72
                                poly(GA) aggregates sequester and impair HR23 and nucleocytoplasmic transport
                                proteins.</a> Nature
                                Neuroscience. 19(5):668-77.

                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;
                                Abelin JG, Patel J, Lu X, Feeney CM, Fagbami L, Creech AL, Hu R, Lam D, Davison
                                D, Pino L, Qiao JW,
                                Kuhn
                                E, Officer A, Li J, Abbatiello S, Subramanian A, Sidman R, Snyder E, Carr SA,
                                Jaffe JD. 2016.<a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/26912667" target="_blank">
                                Reduced-representation
                                phosphosignatures measured by quantitative targeted MS capture cellular states
                                and enable
                                large-scale
                                comparison of drug-induced phenotypes.</a> Molecular and Cellular Proteomics.
                                15(5):1622-41.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Holewinski RJ, Parker SJ, Matlock AD, Venkatraman V, Van Eyk JE. 2016.<a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/26867750" target="_blank"> Methods
                                for SWATH: Data
                                independent acquisition on TripleTOF mass spectrometers.</a> Methods in
                                Molecular Biology.
                                1410:265-79.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Fuller HR, Mandefro B, Shirran SL, Gross AR, Kaus AS, Botting CH, Morris GE,
                                Sareen D. 2016.<a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/26793058" target="_blank"> Spinal
                                muscular atrophy
                                patient
                                iPSC-derived motor neurons have reduced expression of proteins important in
                                neuronal
                                development.</a>
                                Frontiers in Cellular Neuroscience. 9:506.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;
                                van Hasselt JG, Hansen J, Xiong Y, Shim J, Pickard A, Jayaraman G, Hu B, Mahajan
                                M, Gallo J, Sobie
                                EA,
                                Birtwistle MR, Azeloglu EU, Iyengar R. 2016.<a
                                    href="https://www.biorxiv.org/content/early/2016/09/19/075754"
                                    target="_blank">
                                Clinically-weighted
                                transcriptomic signatures for protein kinase inhibitor associated
                                cardiotoxicity.</a> bioRxiv.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Wang Z, Clark NC, Ma'ayan A. 2015.<a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/26048415"
                                    target="_blank"> Dynamics of the discovery process of
                                protein-protein interactions from low content studies.</a> BMC Systems Biology.
                                9(1):26.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;
                                Hornbeck PV, Zhang B, Murray B, Kornhauser JM, Latham V, Skrzypek E. 2015. <a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/25514926" target="_blank">PhosphoSitePlus,
                                2014:
                                mutations, PTMs and recalibrations.</a> Nucleic Acids Research. 43(Database
                                issue):D512-20.
                                <br/>
                                <hr/>
                            </div>

                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Sharma V, Eckels J, Taylor GK, Shulman NJ, Stergachis AB, Joyner SA, Yan P,
                                Whiteaker JR, Halusa GN,
                                Schilling B, Gibson BW, Colangelo CM, Paulovich AG, Carr SA, Jaffe JD, MacCoss
                                MJ, MacLean B.
                                2014.<a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/25102069" target="_blank">
                                Panorama: a targeted
                                proteomics
                                knowledge base.</a> Journal of Proteome Research. 13(9):4205-10.
                                Everley RA, Huttlin EL, Erickson AR, Beausoleil SA, Gygi SP. 2016. <a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/27978624" target="_blank">Neutral
                                loss is a very
                                common
                                occurrence in phosphotyrosine-containing peptides labeled with isobaric
                                tags.</a> Journal of
                                Proteome
                                Research. 16(2):1069-1076.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                McAllister FE, Niepel M, Haas W, Huttlin E, Sorger PK, Gygi SP. 2013. <a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/23607489" target="_blank">Mass
                                spectrometry based
                                method
                                to increase throughput for kinome analyses using ATP probes.</a> Analytical
                                Chemistry.
                                85(9):4666-74.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Luckert K, Gujral TS, Chan M, Sevecka M, Joos TO, Sorger PK, Macbeath G, Pötz O.
                                2012.<a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/22234610" target="_blank"> A dual
                                array-based approach
                                to
                                assess the abundance and posttranslational modification state of signaling
                                proteins.</a> Science
                                Signaling. 5(206):1.
                                <br/>
                                <hr/>
                            </div>


                            <div class="fh5co-sub-lead ">
                                <i class="fa fa-gear" style="color: #3eae40"></i>&nbsp;

                                Kirouac DC, Saez-Rodriguez J, Swantek J, Burke JM, Lauffenburger DA, Sorger PK.
                                2012.<a
                                    href="https://www.ncbi.nlm.nih.gov/pubmed/22548703" target="_blank">
                                Creating and analyzing
                                pathway
                                and protein interaction compendia for modelling signal transduction
                                networks.</a> BMC Systems
                                Biology.
                                6:29.
                                <br/>
                                <hr/>
                            </div>
                        </div>
                    </div>
                    <!--</div>-->
                </div>

            </div>
        </div>
        <div ng-show="showAssaysSubTabVal" style="padding-top: 15px;">

        </div>

        <!--<div ng-show="showCentersSubTabVal" style="padding-top: 15px;">-->
        <!--<div id="fh5co-testimonials2" style="padding-top: 15px;">-->
        <!--&lt;!&ndash;<a name="centersInfo" />&ndash;&gt;-->


        <!--<div class="biggerTab" style="padding-top: 15px;">-->


        <!--&lt;!&ndash;<h2 class="fh5co-lead animate-box text-center">LINCS Proteomics Centers</h2>&ndash;&gt;-->

        <!--<div class="row ">-->
        <!--<div class="col-md-6 col-md-offset-3 text-center">-->

        <!--<h2 class="fh5co-lead " style="font-size: 30px">LINCS Proteomics Data-->
        <!--Centers </h2>-->
        <!--<div class="fh5co-label ">This list includes data generating and data-->
        <!--processing centers-->
        <!--</div>-->
        <!--</div>-->
        <!--</div>-->
        <!--<div class="row">-->
        <!--<div class="col-md-8 col-md-offset-2 text-center animate-box">-->
        <!--<div class="flexslider">-->
        <!--<ul class="slides">-->
        <!--<li>-->
        <!--<blockquote>-->
        <!--<h3>LINCS PCCSE </h3>-->
        <!--<p>The LINCS Proteomic Characterization Center for Signaling-->
        <!--and-->
        <!--Epigenetics at-->
        <!--the-->
        <!--Broad Institute/MIT/University of Washington utilizes-->
        <!--the P100 and-->
        <!--GCP-->
        <!--assays to-->
        <!--profile the response of human cell lines to-->
        <!--perturbations.</p>-->
        <!--<p><cite>&nbsp;</cite></p>-->
        <!--</blockquote>-->
        <!--</li>-->
        <!--<li>-->
        <!--<blockquote>-->
        <!--<h3>LINCS DToxS </h3>-->
        <!--<p>The Drug Toxicity Signature Generation Center is a-->
        <!--Systems-->
        <!--Pharmacology-->
        <!--research-->
        <!--center at the Icahn School of Medicine. The proteomics-->
        <!--experiments-->
        <!--are-->
        <!--conducted at-->
        <!--the Center for Advanced Proteomics, Rutgers-New Jersey-->
        <!--Medical-->
        <!--School.</p>-->
        <!--<p><cite>&nbsp;</cite></p>-->
        <!--</blockquote>-->
        <!--</li>-->
        <!--<li>-->
        <!--<blockquote>-->
        <!--<h3>HMS LINCS </h3>-->
        <!--<p>The HMS LINCS Center collects multiplex-->
        <!--perturbagen-response data-->
        <!--across a-->
        <!--number of complementary-->
        <!--experimental platforms including live- and fixed-cell-->
        <!--imaging,-->
        <!--sandwich-->
        <!--immunoassays, protein mass-->
        <!--spectrometry, and mRNA profiling as a means to measure-->
        <!--the levels,-->
        <!--localization, and states of modification.</p>-->
        <!--<p><cite>&nbsp;</cite></p>-->
        <!--</blockquote>-->
        <!--</li>-->
        <!--<li>-->
        <!--<blockquote>-->
        <!--<h3>NeuroLINCS </h3>-->
        <!--<p>The NeuroLINCS Center studies ALS and SMA by-->
        <!--characterizing the-->
        <!--molecular-->
        <!--networks within-->
        <!--patient-derived iPSCs, and their differentiated progeny,-->
        <!--across many-->
        <!--regulatory layers.</p>-->
        <!--<p><cite>&nbsp;</cite></p>-->
        <!--</blockquote>-->
        <!--</li>-->
        <!--<li>-->
        <!--<blockquote>-->
        <!--<h3>BD2K-LINCS DCIC </h3>-->
        <!--<p>The BD2K-LINCS Data Coordination and Integration Centers-->
        <!--at Icahn-->
        <!--School of-->
        <!--Medicine, university of Miami-->
        <!--and university of Cincinnati are responsible for-->
        <!--harmonizing-->
        <!--LINCS data with other available resources.</p>-->
        <!--<p><cite>&nbsp;</cite></p>-->
        <!--</blockquote>-->
        <!--</li>-->
        <!--</ul>-->
        <!--</div>-->
        <!--<div class="flexslider-controls" style="padding-top: 10px">-->
        <!--<ol class="flex-control-nav">-->
        <!--<li class="animate-box"><img src="/lincsproteomics/images/labkey-logo2.png"-->
        <!--alt="Free HTML5 Bootstrap Template by FREEHTML5.co">-->
        <!--</li>-->
        <!--<li class="animate-box"><img src="/lincsproteomics/images/dtoxs2.png"-->
        <!--alt="Free HTML5 Bootstrap Template by FREEHTML5.co">-->
        <!--</li>-->
        <!--<li class="animate-box"><img src="/lincsproteomics/images/hms-lincs2.png"-->
        <!--alt="Free HTML5 Bootstrap Template by FREEHTML5.co">-->
        <!--</li>-->
        <!--<li class="animate-box"><img src="/lincsproteomics/images/neurolincs2.jpeg"-->
        <!--alt="Free HTML5 Bootstrap Template by FREEHTML5.co">-->
        <!--</li>-->
        <!--<li class="animate-box"><img src="/lincsproteomics/images/dcic3.png"-->
        <!--alt="Free HTML5 Bootstrap Template by FREEHTML5.co">-->
        <!--</li>-->
        <!--</ol>-->
        <!--</div>-->
        <!--</div>-->
        <!--</div>-->
        <!--</div>-->
        <!--</div>-->

        <!--&lt;!&ndash;<div id="fh5co-pricing">&ndash;&gt;-->
        <!--<div class="biggerTab" style="padding-top: 30px">-->

        <!--<iframe src="https://www.google.com/maps/d/embed?mid=1MhDhNDsvCJvAmSNypmyCaj-g2EQ&hl=en"-->
        <!--width="100%"-->
        <!--height="700"></iframe>-->
        <!--&lt;!&ndash;</p>&ndash;&gt;-->
        <!--</div>-->
        <!--</div>-->


    </div>

    <!-- END #fh5co-about -->


    <!--<section id="fh5co-features-2" style="padding: 20px 0px 10px 0px">-->


    <!---->
    <!--</section>-->


    <!--<section id="fh5co-subscribe" style="padding: 60px 5px 5px 5px">-->
    <!--<hr/>-->
    <!--<div class="container">-->

    <!--</div>-->
    <!--</section>-->

    <!--<hr/>-->


    <!--<section id="fh5co-projects2" style="padding: 30px 0px 0px 0px">-->

    <!--<div class="container">-->


    <!--</div>-->

    <!--</section>-->

    <!--<div class="container">-->


    <!--<div class="row">-->
    <!--<div class="col-sm-3">-->
    <!--<figure class="animate-box">-->
    <!--<img src="/lincsproteomics/images/pinet-logo.png" alt="Free HTML5 Bootstrap Template by FREEHTML5.co"-->
    <!--class="img-responsive" width="300" height="300">-->
    <!--</figure>-->
    <!--</div>-->
    <!--<div class="col-sm-9">-->
    <!--<div class="fh5co-label animate-box">More about piNET</div>-->
    <!--<h2 class="fh5co-lead animate-box">piNET</h2>-->
    <!--<p class="fh5co-sub-lead animate-box"><a href="http://eh3.uc.edu/pinet/" target="_blank">-->
    <!--<span style="color:blue">&#10004;</span> piNet</a> can be used to annotate, map and-->
    <!--analyze-->
    <!--a set of peptide moieties-->
    <!--(including post-translationally modified peptides) identified by either-->
    <!--targeted (SRM type) or SWATH (discovery) proteomic assays, or another-->
    <!--related mass spectrometry based protein moieties identification and-->
    <!--quantification technique. Starting from a set of peptides, and optional-->
    <!--modifications within those peptides, piNET provides mapping into-->
    <!--canonical UniProt protein entries, standardized annotation of-->
    <!--modifications using PsiMOD ontology, and mapping of the sites of PTMs in-->
    <!--the full UniProt entries. piNET further generates Protein Line Notation-->
    <!--tokens to represent each peptide moiety with modifications for automated-->
    <!--annotation exchange systems. The set of proteins and PTM sites can be-->
    <!--further projected onto biological pathways using enrichment analysis and-->
    <!--a comprehensive library of biological gene/protein sets through Enrichr.-->
    <!--For phosphorylation, PhosphoSitePlus known kinase - target peptides (and-->
    <!--extrapolations form known examples to sufficiently similar putative-->
    <!--target peptides) are used to provide a site specific network and-->
    <!--pathways views. Work is in progress to extend the capabilities to-->
    <!--include other PTMs in network projections, and to enable annotation of-->
    <!--mouse and rat proteomics data.</p>-->
    <!--</div>-->
    <!--</div>-->


    <!--<div class="row">-->
    <!--<div class="col-sm-3">-->
    <!--<figure class="animate-box">-->
    <!--<img src="/lincsproteomics/images/pilincs.jpg" alt="Free HTML5 Bootstrap Template by FREEHTML5.co"-->
    <!--class="img-responsive" width="350" height="350">-->
    <!--</figure>-->
    <!--</div>-->
    <!--<div class="col-sm-9">-->
    <!--<div class="fh5co-label animate-box">More about piLINCS</div>-->
    <!--<h2 class="fh5co-lead animate-box">piLINCS</h2>-->
    <!--<p class="fh5co-sub-lead animate-box"><a href="http://eh3.uc.edu/pilincs/#/"-->
    <!--target="_blank">-->
    <!--<span style="color:blue">&#10004;</span> piLINCS</a> has been developed as part of the-->
    <!--BD2K-LINCS Data Coordination and Integration Center-->
    <!--to provide interactive and API access to proteomics data generated by the Library of-->
    <!--Integrated-->
    <!--Network-based Signatures (LINCS) program. At present, piLINCS enables access to the-->
    <!--reduced-->
    <!--representation phosphoproteomic ("P100") and global chromatin ("GCP") profiles generated-->
    <!--by-->
    <!--using-->
    <!--mass spectrometry techniques to characterize proteome level molecular signatures of-->
    <!--(responses to)-->
    <!--small molecule and genetic perturbations in a number of different cell lines-->
    <!--(for details see LINCS/Broad). These different profiles (obtained by merging QC+NORM-->
    <!--level-->
    <!--processed GCT files from Panorama repository) can be browsed interactively, filtered-->
    <!--based-->
    <!--on user-->
    <!--defined selection of cell - perturbation - dose tuples, merged across different profiles-->
    <!--(for the-->
    <!--same tuples), analyzed in iLINCS (more tools to come), and exported in several different-->
    <!--formats.</p>-->
    <!--</div>-->
    <!--</div>-->


    <!--</div>-->


    <!-- END #fh5co-subscribe -->


    <!--<a name="foo">-->
    <!--ggggg-->
    <!--</a>-->

    <!--<div class="container">-->

    <!--<div ng-view></div>-->

    <!--</div>-->

</div>

